{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "import glob\n",
    "import gzip\n",
    "\n",
    "\n",
    "#snp_out_dir='/nrnb/users/btsui/Data/tcga_extracted_lgg_snp/'\n",
    "#/nrnb/users/btsui/Data/all_seq/snp/\n",
    "preprocessed_vcf_dir='/nrnb/users/btsui/Data/all_seq/snp/'\n",
    "\n",
    "\n",
    "inFnames=pd.Series(os.listdir(preprocessed_vcf_dir))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "outDir='./TCGA_VCF.pickle'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "#vcfDirs=glob.glob('/nrnb/users/btsui/Data/tcga_orig_vcf/**/*.vcf$')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 183,
   "metadata": {},
   "outputs": [],
   "source": [
    "vcfDirs=glob.glob('/nrnb/users/btsui/Data/tcga_orig_vcf/**/*.vcf*')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#len(vcfDirs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 185,
   "metadata": {},
   "outputs": [],
   "source": [
    "from tqdm import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "  0%|          | 0/2119 [00:00<?, ?it/s]\u001b[A\n",
      "  0%|          | 8/2119 [00:00<00:28, 75.32it/s]\u001b[A\n",
      "  1%|          | 16/2119 [00:00<00:28, 73.72it/s]\u001b[A\n",
      "  1%|          | 22/2119 [00:00<00:30, 69.34it/s]\u001b[A\n",
      "  1%|▏         | 29/2119 [00:00<00:30, 68.85it/s]\u001b[A\n",
      "  2%|▏         | 38/2119 [00:00<00:29, 71.61it/s]\u001b[A\n",
      "  2%|▏         | 46/2119 [00:00<00:28, 71.53it/s]\u001b[A\n",
      "  3%|▎         | 53/2119 [00:00<00:34, 59.85it/s]\u001b[A\n",
      "  3%|▎         | 62/2119 [00:00<00:32, 62.91it/s]\u001b[A\n",
      "  3%|▎         | 62/2119 [00:14<07:47,  4.40it/s]\u001b[A\n",
      "  3%|▎         | 69/2119 [00:28<13:53,  2.46it/s]\u001b[A\n",
      "  4%|▎         | 76/2119 [00:28<12:55,  2.63it/s]\u001b[A\n",
      "  4%|▎         | 77/2119 [00:29<12:52,  2.64it/s]\u001b[A\n",
      "  4%|▍         | 86/2119 [00:29<11:30,  2.94it/s]\u001b[A\n",
      "  4%|▍         | 95/2119 [00:29<10:24,  3.24it/s]\u001b[A\n",
      "  5%|▍         | 100/2119 [00:33<11:12,  3.00it/s]\u001b[A\n",
      "  5%|▍         | 104/2119 [00:37<12:01,  2.79it/s]\u001b[A\n",
      "  5%|▌         | 112/2119 [00:37<11:09,  3.00it/s]\u001b[A\n",
      "  5%|▌         | 116/2119 [00:37<10:49,  3.09it/s]\u001b[A\n",
      "  6%|▌         | 120/2119 [00:37<10:29,  3.18it/s]\u001b[A\n",
      "  6%|▌         | 126/2119 [00:37<09:59,  3.33it/s]\u001b[A\n",
      "  6%|▌         | 132/2119 [00:38<09:32,  3.47it/s]\u001b[A\n",
      "  6%|▋         | 136/2119 [00:38<09:26,  3.50it/s]\u001b[A\n",
      "  7%|▋         | 143/2119 [00:41<09:35,  3.44it/s]\u001b[A\n",
      "  7%|▋         | 147/2119 [00:41<09:22,  3.51it/s]\u001b[A\n",
      "  7%|▋         | 151/2119 [00:42<09:09,  3.58it/s]\u001b[A\n",
      "  8%|▊         | 162/2119 [00:42<08:30,  3.83it/s]\u001b[A\n",
      "  8%|▊         | 168/2119 [00:42<08:13,  3.95it/s]\u001b[A\n",
      "  8%|▊         | 172/2119 [00:53<10:05,  3.22it/s]\u001b[A\n",
      "  8%|▊         | 173/2119 [00:55<10:19,  3.14it/s]\u001b[A\n",
      "  8%|▊         | 175/2119 [00:58<10:49,  3.00it/s]\u001b[A\n",
      "  9%|▊         | 182/2119 [00:58<10:22,  3.11it/s]\u001b[A\n",
      "  9%|▉         | 188/2119 [00:59<10:08,  3.17it/s]\u001b[A\n",
      "  9%|▉         | 191/2119 [01:04<10:55,  2.94it/s]\u001b[A\n",
      "  9%|▉         | 198/2119 [01:05<10:31,  3.04it/s]\u001b[A\n",
      " 10%|▉         | 207/2119 [01:05<10:02,  3.18it/s]\u001b[A\n",
      " 10%|█         | 212/2119 [01:16<11:32,  2.75it/s]\u001b[A\n",
      " 10%|█         | 214/2119 [01:17<11:33,  2.75it/s]\u001b[A\n",
      " 11%|█         | 224/2119 [01:17<10:59,  2.87it/s]\u001b[A\n",
      " 11%|█         | 231/2119 [01:18<10:38,  2.96it/s]\u001b[A\n",
      " 11%|█         | 236/2119 [01:28<11:46,  2.67it/s]\u001b[A\n",
      " 11%|█         | 238/2119 [01:30<11:53,  2.64it/s]\u001b[A\n",
      " 11%|█▏        | 242/2119 [01:34<12:13,  2.56it/s]\u001b[A\n",
      " 12%|█▏        | 252/2119 [01:34<11:41,  2.66it/s]\u001b[A\n",
      " 12%|█▏        | 256/2119 [01:42<12:29,  2.49it/s]\u001b[A\n",
      " 12%|█▏        | 261/2119 [01:43<12:15,  2.53it/s]\u001b[A\n",
      " 12%|█▏        | 261/2119 [01:54<13:32,  2.29it/s]\u001b[A\n",
      " 12%|█▏        | 263/2119 [01:56<13:41,  2.26it/s]\u001b[A\n",
      " 12%|█▏        | 264/2119 [02:01<14:11,  2.18it/s]\u001b[A\n",
      " 13%|█▎        | 265/2119 [02:04<14:34,  2.12it/s]\u001b[A\n",
      " 13%|█▎        | 266/2119 [02:09<15:02,  2.05it/s]\u001b[A\n",
      " 13%|█▎        | 267/2119 [02:09<15:01,  2.06it/s]\u001b[A\n",
      " 13%|█▎        | 270/2119 [02:14<15:22,  2.00it/s]\u001b[A\n",
      " 13%|█▎        | 271/2119 [02:15<15:20,  2.01it/s]\u001b[A\n",
      " 13%|█▎        | 272/2119 [02:20<15:53,  1.94it/s]\u001b[A\n",
      " 13%|█▎        | 273/2119 [02:20<15:50,  1.94it/s]\u001b[A\n",
      " 13%|█▎        | 275/2119 [02:20<15:44,  1.95it/s]\u001b[A\n",
      " 13%|█▎        | 276/2119 [02:25<16:12,  1.89it/s]\u001b[A\n",
      " 13%|█▎        | 279/2119 [02:25<16:02,  1.91it/s]\u001b[A\n",
      " 13%|█▎        | 280/2119 [02:27<16:10,  1.89it/s]\u001b[A\n",
      " 13%|█▎        | 285/2119 [02:28<15:53,  1.92it/s]\u001b[A\n",
      " 13%|█▎        | 286/2119 [02:30<16:02,  1.90it/s]\u001b[A\n",
      " 14%|█▎        | 288/2119 [02:32<16:10,  1.89it/s]\u001b[A\n",
      " 14%|█▎        | 289/2119 [02:37<16:39,  1.83it/s]\u001b[A\n",
      " 14%|█▎        | 290/2119 [02:44<17:20,  1.76it/s]\u001b[A\n",
      " 14%|█▎        | 291/2119 [02:59<18:44,  1.63it/s]\u001b[A\n",
      " 14%|█▍        | 294/2119 [02:59<18:33,  1.64it/s]\u001b[A\n",
      " 14%|█▍        | 295/2119 [02:59<18:29,  1.64it/s]\u001b[A\n",
      " 14%|█▍        | 300/2119 [02:59<18:08,  1.67it/s]\u001b[A\n",
      " 14%|█▍        | 304/2119 [03:09<18:51,  1.60it/s]\u001b[A\n",
      " 14%|█▍        | 306/2119 [03:10<18:47,  1.61it/s]\u001b[A\n",
      " 15%|█▍        | 308/2119 [03:10<18:42,  1.61it/s]\u001b[A\n",
      " 15%|█▍        | 310/2119 [03:11<18:37,  1.62it/s]\u001b[A\n",
      " 15%|█▍        | 313/2119 [03:12<18:28,  1.63it/s]\u001b[A\n",
      " 15%|█▍        | 314/2119 [03:22<19:23,  1.55it/s]\u001b[A\n",
      " 15%|█▍        | 317/2119 [03:22<19:11,  1.56it/s]\u001b[A\n",
      " 15%|█▌        | 320/2119 [03:22<18:59,  1.58it/s]\u001b[A\n",
      " 15%|█▌        | 322/2119 [03:30<19:34,  1.53it/s]\u001b[A\n",
      " 15%|█▌        | 323/2119 [03:30<19:31,  1.53it/s]\u001b[A\n",
      " 15%|█▌        | 324/2119 [03:36<19:59,  1.50it/s]\u001b[A\n",
      " 15%|█▌        | 325/2119 [03:40<20:17,  1.47it/s]\u001b[A\n",
      " 15%|█▌        | 326/2119 [03:48<20:56,  1.43it/s]\u001b[A\n",
      " 15%|█▌        | 328/2119 [03:59<21:46,  1.37it/s]\u001b[A\n",
      " 16%|█▌        | 331/2119 [04:19<23:22,  1.27it/s]\u001b[A\n",
      " 16%|█▌        | 332/2119 [04:23<23:38,  1.26it/s]\u001b[A\n",
      " 16%|█▌        | 334/2119 [04:26<23:44,  1.25it/s]\u001b[A\n",
      " 16%|█▌        | 335/2119 [04:26<23:40,  1.26it/s]\u001b[A\n",
      " 16%|█▌        | 336/2119 [04:31<23:58,  1.24it/s]\u001b[A\n",
      " 16%|█▌        | 337/2119 [04:37<24:25,  1.22it/s]\u001b[A\n",
      " 16%|█▌        | 338/2119 [04:42<24:46,  1.20it/s]\u001b[A\n",
      " 16%|█▌        | 339/2119 [04:44<24:53,  1.19it/s]\u001b[A\n",
      " 16%|█▌        | 340/2119 [04:49<25:15,  1.17it/s]\u001b[A\n",
      " 16%|█▌        | 341/2119 [04:54<25:36,  1.16it/s]\u001b[A\n",
      " 16%|█▌        | 342/2119 [05:02<26:13,  1.13it/s]\u001b[A\n",
      " 16%|█▌        | 343/2119 [05:10<26:49,  1.10it/s]\u001b[A\n",
      " 16%|█▋        | 348/2119 [05:18<26:59,  1.09it/s]\u001b[A\n",
      " 17%|█▋        | 350/2119 [05:18<26:48,  1.10it/s]\u001b[A\n",
      " 17%|█▋        | 352/2119 [05:18<26:38,  1.11it/s]\u001b[A\n",
      " 17%|█▋        | 355/2119 [05:20<26:33,  1.11it/s]\u001b[A\n",
      " 17%|█▋        | 358/2119 [05:27<26:52,  1.09it/s]\u001b[A\n",
      " 17%|█▋        | 359/2119 [05:32<27:08,  1.08it/s]\u001b[A\n",
      " 17%|█▋        | 360/2119 [05:32<27:04,  1.08it/s]\u001b[A\n",
      " 17%|█▋        | 361/2119 [05:38<27:30,  1.07it/s]\u001b[A\n",
      " 17%|█▋        | 364/2119 [05:50<28:11,  1.04it/s]\u001b[A\n",
      " 17%|█▋        | 366/2119 [05:56<28:26,  1.03it/s]\u001b[A\n",
      " 17%|█▋        | 367/2119 [05:56<28:21,  1.03it/s]\u001b[A\n",
      " 17%|█▋        | 368/2119 [05:56<28:18,  1.03it/s]\u001b[A\n",
      " 17%|█▋        | 370/2119 [06:02<28:33,  1.02it/s]\u001b[A\n",
      " 18%|█▊        | 372/2119 [06:09<28:55,  1.01it/s]\u001b[A\n",
      " 18%|█▊        | 373/2119 [06:10<28:53,  1.01it/s]\u001b[A\n",
      " 18%|█▊        | 374/2119 [06:10<28:49,  1.01it/s]\u001b[A\n",
      " 18%|█▊        | 375/2119 [06:17<29:13,  1.01s/it]\u001b[A\n",
      " 18%|█▊        | 376/2119 [06:26<29:53,  1.03s/it]\u001b[A\n",
      " 18%|█▊        | 377/2119 [06:27<29:50,  1.03s/it]\u001b[A\n",
      " 18%|█▊        | 378/2119 [06:27<29:46,  1.03s/it]\u001b[A\n",
      " 18%|█▊        | 380/2119 [06:28<29:37,  1.02s/it]\u001b[A\n",
      " 18%|█▊        | 381/2119 [06:36<30:08,  1.04s/it]\u001b[A\n",
      " 18%|█▊        | 382/2119 [06:37<30:07,  1.04s/it]\u001b[A\n",
      " 18%|█▊        | 383/2119 [06:37<30:02,  1.04s/it]\u001b[A\n",
      " 18%|█▊        | 388/2119 [06:42<29:55,  1.04s/it]\u001b[A\n",
      " 18%|█▊        | 390/2119 [06:43<29:46,  1.03s/it]\u001b[A\n",
      " 18%|█▊        | 392/2119 [06:43<29:37,  1.03s/it]\u001b[A\n",
      " 19%|█▊        | 396/2119 [06:46<29:27,  1.03s/it]\u001b[A\n",
      " 19%|█▉        | 398/2119 [06:51<29:38,  1.03s/it]\u001b[A\n",
      " 19%|█▉        | 401/2119 [06:51<29:23,  1.03s/it]\u001b[A\n",
      " 19%|█▉        | 403/2119 [06:51<29:13,  1.02s/it]\u001b[A\n",
      " 19%|█▉        | 405/2119 [06:52<29:07,  1.02s/it]\u001b[A\n",
      " 19%|█▉        | 406/2119 [06:56<29:15,  1.02s/it]\u001b[A\n",
      " 19%|█▉        | 411/2119 [06:57<28:52,  1.01s/it]\u001b[A\n",
      " 20%|█▉        | 415/2119 [06:57<28:33,  1.01s/it]\u001b[A\n",
      " 20%|█▉        | 416/2119 [06:57<28:30,  1.00s/it]\u001b[A\n",
      " 20%|█▉        | 417/2119 [06:58<28:29,  1.00s/it]\u001b[A\n",
      " 20%|█▉        | 419/2119 [07:02<28:35,  1.01s/it]\u001b[A\n",
      " 20%|█▉        | 420/2119 [07:03<28:32,  1.01s/it]\u001b[A\n",
      " 20%|█▉        | 422/2119 [07:03<28:22,  1.00s/it]\u001b[A\n",
      " 20%|█▉        | 423/2119 [07:14<29:00,  1.03s/it]\u001b[A\n",
      " 20%|██        | 424/2119 [07:14<28:55,  1.02s/it]\u001b[A\n",
      " 20%|██        | 425/2119 [07:16<29:00,  1.03s/it]\u001b[A\n",
      " 20%|██        | 426/2119 [07:21<29:16,  1.04s/it]\u001b[A\n",
      " 20%|██        | 427/2119 [07:27<29:31,  1.05s/it]\u001b[A\n",
      " 20%|██        | 429/2119 [07:30<29:33,  1.05s/it]\u001b[A\n",
      " 20%|██        | 431/2119 [07:30<29:24,  1.05s/it]\u001b[A\n",
      " 20%|██        | 432/2119 [07:30<29:20,  1.04s/it]\u001b[A\n",
      " 20%|██        | 434/2119 [07:31<29:11,  1.04s/it]\u001b[A\n",
      " 21%|██        | 435/2119 [07:31<29:07,  1.04s/it]\u001b[A\n",
      " 21%|██        | 436/2119 [07:37<29:25,  1.05s/it]\u001b[A\n",
      " 21%|██        | 439/2119 [07:37<29:11,  1.04s/it]\u001b[A\n",
      " 21%|██        | 444/2119 [07:37<28:47,  1.03s/it]\u001b[A\n",
      " 21%|██        | 445/2119 [07:40<28:53,  1.04s/it]\u001b[A\n",
      " 21%|██        | 449/2119 [07:47<28:57,  1.04s/it]\u001b[A\n",
      " 21%|██        | 450/2119 [07:47<28:53,  1.04s/it]\u001b[A\n",
      " 21%|██▏       | 452/2119 [07:49<28:50,  1.04s/it]\u001b[A\n",
      " 22%|██▏       | 456/2119 [07:51<28:39,  1.03s/it]\u001b[A\n",
      " 22%|██▏       | 458/2119 [07:58<28:54,  1.04s/it]\u001b[A\n",
      " 22%|██▏       | 459/2119 [07:59<28:52,  1.04s/it]\u001b[A\n",
      " 22%|██▏       | 461/2119 [08:01<28:51,  1.04s/it]\u001b[A\n",
      " 22%|██▏       | 462/2119 [08:09<29:14,  1.06s/it]\u001b[A\n",
      " 22%|██▏       | 464/2119 [08:17<29:35,  1.07s/it]\u001b[A\n",
      " 22%|██▏       | 465/2119 [08:17<29:30,  1.07s/it]\u001b[A\n",
      " 22%|██▏       | 466/2119 [08:18<29:26,  1.07s/it]\u001b[A\n",
      " 22%|██▏       | 468/2119 [08:18<29:17,  1.06s/it]\u001b[A\n",
      " 22%|██▏       | 469/2119 [08:18<29:14,  1.06s/it]\u001b[A\n",
      " 22%|██▏       | 470/2119 [08:19<29:10,  1.06s/it]\u001b[A\n",
      " 22%|██▏       | 471/2119 [08:26<29:30,  1.07s/it]\u001b[A\n",
      " 22%|██▏       | 473/2119 [08:26<29:22,  1.07s/it]\u001b[A\n",
      " 22%|██▏       | 476/2119 [08:31<29:24,  1.07s/it]\u001b[A\n",
      " 23%|██▎       | 477/2119 [08:37<29:40,  1.08s/it]\u001b[A\n",
      " 23%|██▎       | 479/2119 [08:37<29:32,  1.08s/it]\u001b[A\n",
      " 23%|██▎       | 480/2119 [08:45<29:55,  1.10s/it]\u001b[A\n",
      " 23%|██▎       | 484/2119 [08:49<29:48,  1.09s/it]\u001b[A\n",
      " 23%|██▎       | 488/2119 [08:53<29:42,  1.09s/it]\u001b[A\n",
      " 23%|██▎       | 490/2119 [08:59<29:53,  1.10s/it]\u001b[A\n",
      " 23%|██▎       | 491/2119 [09:00<29:50,  1.10s/it]\u001b[A\n",
      " 23%|██▎       | 496/2119 [09:03<29:39,  1.10s/it]\u001b[A\n",
      " 23%|██▎       | 497/2119 [09:10<29:58,  1.11s/it]\u001b[A\n",
      " 24%|██▎       | 498/2119 [09:19<30:19,  1.12s/it]\u001b[A\n",
      " 24%|██▎       | 502/2119 [09:40<31:10,  1.16s/it]\u001b[A\n",
      " 24%|██▍       | 507/2119 [09:46<31:04,  1.16s/it]\u001b[A\n",
      " 24%|██▍       | 508/2119 [09:54<31:24,  1.17s/it]\u001b[A\n",
      " 24%|██▍       | 511/2119 [10:01<31:34,  1.18s/it]\u001b[A\n",
      " 24%|██▍       | 512/2119 [10:10<31:56,  1.19s/it]\u001b[A\n",
      " 24%|██▍       | 515/2119 [10:15<31:56,  1.20s/it]\u001b[A\n",
      " 24%|██▍       | 517/2119 [10:16<31:49,  1.19s/it]\u001b[A\n",
      " 24%|██▍       | 518/2119 [10:16<31:45,  1.19s/it]\u001b[A\n",
      " 24%|██▍       | 519/2119 [10:16<31:40,  1.19s/it]\u001b[A\n",
      " 25%|██▍       | 521/2119 [10:22<31:47,  1.19s/it]\u001b[A\n",
      " 25%|██▍       | 522/2119 [10:28<32:02,  1.20s/it]\u001b[A\n",
      " 25%|██▍       | 523/2119 [10:36<32:23,  1.22s/it]\u001b[A\n",
      " 25%|██▍       | 524/2119 [10:37<32:19,  1.22s/it]\u001b[A\n",
      " 25%|██▍       | 525/2119 [10:40<32:24,  1.22s/it]\u001b[A\n",
      " 25%|██▍       | 526/2119 [10:40<32:20,  1.22s/it]\u001b[A\n",
      " 25%|██▍       | 527/2119 [10:47<32:36,  1.23s/it]\u001b[A\n",
      " 25%|██▌       | 530/2119 [10:52<32:35,  1.23s/it]\u001b[A\n",
      " 25%|██▌       | 531/2119 [10:52<32:31,  1.23s/it]\u001b[A\n",
      " 25%|██▌       | 533/2119 [10:52<32:21,  1.22s/it]\u001b[A\n",
      " 25%|██▌       | 539/2119 [10:52<31:53,  1.21s/it]\u001b[A\n",
      " 26%|██▌       | 541/2119 [10:52<31:44,  1.21s/it]\u001b[A\n",
      " 26%|██▌       | 546/2119 [10:57<31:35,  1.21s/it]\u001b[A\n",
      " 26%|██▌       | 550/2119 [10:58<31:18,  1.20s/it]\u001b[A\n",
      " 26%|██▌       | 551/2119 [11:07<31:38,  1.21s/it]\u001b[A\n",
      " 26%|██▌       | 553/2119 [11:15<31:52,  1.22s/it]\u001b[A\n",
      " 26%|██▋       | 557/2119 [11:24<32:00,  1.23s/it]\u001b[A\n",
      " 26%|██▋       | 560/2119 [11:25<31:47,  1.22s/it]\u001b[A\n",
      " 26%|██▋       | 561/2119 [11:25<31:43,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 564/2119 [11:25<31:30,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 566/2119 [11:25<31:22,  1.21s/it]\u001b[A\n",
      " 27%|██▋       | 567/2119 [11:26<31:18,  1.21s/it]\u001b[A\n",
      " 27%|██▋       | 568/2119 [11:33<31:34,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 570/2119 [11:34<31:26,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 572/2119 [11:42<31:39,  1.23s/it]\u001b[A\n",
      " 27%|██▋       | 574/2119 [11:42<31:30,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 576/2119 [11:42<31:22,  1.22s/it]\u001b[A\n",
      " 27%|██▋       | 579/2119 [11:42<31:09,  1.21s/it]\u001b[A\n",
      " 28%|██▊       | 583/2119 [11:51<31:15,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 584/2119 [11:52<31:11,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 586/2119 [11:52<31:03,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 588/2119 [11:55<31:04,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 589/2119 [11:56<31:00,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 590/2119 [12:01<31:10,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 593/2119 [12:02<30:58,  1.22s/it]\u001b[A\n",
      " 28%|██▊       | 594/2119 [12:15<31:28,  1.24s/it]\u001b[A\n",
      " 28%|██▊       | 596/2119 [12:15<31:20,  1.23s/it]\u001b[A\n",
      " 28%|██▊       | 597/2119 [12:16<31:16,  1.23s/it]\u001b[A\n",
      " 28%|██▊       | 598/2119 [12:16<31:13,  1.23s/it]\u001b[A\n",
      " 28%|██▊       | 601/2119 [12:17<31:02,  1.23s/it]\u001b[A\n",
      " 28%|██▊       | 602/2119 [12:18<31:00,  1.23s/it]\u001b[A\n",
      " 28%|██▊       | 603/2119 [12:18<30:57,  1.22s/it]\u001b[A\n",
      " 29%|██▊       | 604/2119 [12:18<30:53,  1.22s/it]\u001b[A\n",
      " 29%|██▊       | 607/2119 [12:19<30:41,  1.22s/it]\u001b[A\n",
      " 29%|██▊       | 609/2119 [12:31<31:02,  1.23s/it]\u001b[A\n",
      " 29%|██▉       | 610/2119 [12:37<31:13,  1.24s/it]\u001b[A\n",
      " 29%|██▉       | 612/2119 [12:37<31:05,  1.24s/it]\u001b[A\n",
      " 29%|██▉       | 615/2119 [12:45<31:12,  1.24s/it]\u001b[A\n",
      " 29%|██▉       | 616/2119 [12:55<31:31,  1.26s/it]\u001b[A\n",
      " 29%|██▉       | 617/2119 [12:55<31:28,  1.26s/it]\u001b[A\n",
      " 29%|██▉       | 618/2119 [12:56<31:25,  1.26s/it]\u001b[A\n",
      " 29%|██▉       | 622/2119 [12:56<31:09,  1.25s/it]\u001b[A\n",
      " 29%|██▉       | 623/2119 [13:00<31:14,  1.25s/it]\u001b[A\n",
      " 29%|██▉       | 625/2119 [13:12<31:33,  1.27s/it]\u001b[A\n",
      " 30%|██▉       | 627/2119 [13:12<31:26,  1.26s/it]\u001b[A\n",
      " 30%|██▉       | 628/2119 [13:12<31:22,  1.26s/it]\u001b[A\n",
      " 30%|██▉       | 629/2119 [13:13<31:19,  1.26s/it]\u001b[A\n",
      " 30%|██▉       | 630/2119 [13:13<31:15,  1.26s/it]\u001b[A\n",
      " 30%|██▉       | 632/2119 [13:22<31:27,  1.27s/it]\u001b[A\n",
      " 30%|██▉       | 635/2119 [13:22<31:15,  1.26s/it]\u001b[A\n",
      " 30%|███       | 636/2119 [13:30<31:28,  1.27s/it]\u001b[A\n",
      " 30%|███       | 637/2119 [13:34<31:35,  1.28s/it]\u001b[A\n",
      " 30%|███       | 638/2119 [13:34<31:31,  1.28s/it]\u001b[A\n",
      " 30%|███       | 639/2119 [13:38<31:36,  1.28s/it]\u001b[A\n",
      " 30%|███       | 641/2119 [13:40<31:31,  1.28s/it]\u001b[A\n",
      " 30%|███       | 642/2119 [13:40<31:28,  1.28s/it]\u001b[A\n",
      " 30%|███       | 644/2119 [13:41<31:20,  1.28s/it]\u001b[A\n",
      " 30%|███       | 645/2119 [13:42<31:18,  1.27s/it]\u001b[A\n",
      " 30%|███       | 646/2119 [13:42<31:14,  1.27s/it]\u001b[A\n",
      " 31%|███       | 647/2119 [13:43<31:13,  1.27s/it]\u001b[A\n",
      " 31%|███       | 649/2119 [13:43<31:06,  1.27s/it]\u001b[A\n",
      " 31%|███       | 653/2119 [13:44<30:50,  1.26s/it]\u001b[A\n",
      " 31%|███       | 654/2119 [13:45<30:48,  1.26s/it]\u001b[A\n",
      " 31%|███       | 659/2119 [13:47<30:33,  1.26s/it]\u001b[A\n",
      " 31%|███       | 660/2119 [13:49<30:33,  1.26s/it]\u001b[A\n",
      " 31%|███       | 662/2119 [13:51<30:31,  1.26s/it]\u001b[A\n",
      " 31%|███▏      | 663/2119 [13:53<30:29,  1.26s/it]\u001b[A\n",
      " 31%|███▏      | 664/2119 [13:59<30:40,  1.27s/it]\u001b[A\n",
      " 31%|███▏      | 666/2119 [14:04<30:43,  1.27s/it]\u001b[A\n",
      " 31%|███▏      | 667/2119 [14:11<30:53,  1.28s/it]\u001b[A\n",
      " 32%|███▏      | 669/2119 [14:11<30:45,  1.27s/it]\u001b[A\n",
      " 32%|███▏      | 671/2119 [14:16<30:47,  1.28s/it]\u001b[A\n",
      " 32%|███▏      | 672/2119 [14:21<30:54,  1.28s/it]\u001b[A\n",
      " 32%|███▏      | 673/2119 [14:25<30:59,  1.29s/it]\u001b[A\n",
      " 32%|███▏      | 674/2119 [14:32<31:10,  1.29s/it]\u001b[A\n",
      " 32%|███▏      | 675/2119 [14:32<31:07,  1.29s/it]\u001b[A\n",
      " 32%|███▏      | 676/2119 [14:46<31:32,  1.31s/it]\u001b[A\n",
      " 32%|███▏      | 678/2119 [14:57<31:47,  1.32s/it]\u001b[A\n",
      " 32%|███▏      | 679/2119 [15:00<31:50,  1.33s/it]\u001b[A\n",
      " 32%|███▏      | 681/2119 [15:00<31:42,  1.32s/it]\u001b[A\n",
      " 32%|███▏      | 685/2119 [15:05<31:36,  1.32s/it]\u001b[A\n",
      " 32%|███▏      | 686/2119 [15:11<31:44,  1.33s/it]\u001b[A\n",
      " 32%|███▏      | 688/2119 [15:16<31:46,  1.33s/it]\u001b[A\n",
      " 33%|███▎      | 689/2119 [15:16<31:42,  1.33s/it]\u001b[A\n",
      " 33%|███▎      | 691/2119 [15:24<31:50,  1.34s/it]\u001b[A\n",
      " 33%|███▎      | 696/2119 [15:34<31:50,  1.34s/it]\u001b[A\n",
      " 33%|███▎      | 697/2119 [15:41<32:00,  1.35s/it]\u001b[A\n",
      " 33%|███▎      | 701/2119 [15:49<32:00,  1.35s/it]\u001b[A\n",
      " 33%|███▎      | 702/2119 [15:55<32:07,  1.36s/it]\u001b[A\n",
      " 33%|███▎      | 703/2119 [15:55<32:04,  1.36s/it]\u001b[A\n",
      " 33%|███▎      | 704/2119 [15:58<32:07,  1.36s/it]\u001b[A\n",
      " 33%|███▎      | 706/2119 [15:59<31:59,  1.36s/it]\u001b[A\n",
      " 33%|███▎      | 707/2119 [16:06<32:09,  1.37s/it]\u001b[A\n",
      " 33%|███▎      | 708/2119 [16:06<32:06,  1.37s/it]\u001b[A\n",
      " 34%|███▎      | 711/2119 [16:17<32:16,  1.38s/it]\u001b[A\n",
      " 34%|███▎      | 713/2119 [16:18<32:09,  1.37s/it]\u001b[A\n",
      " 34%|███▎      | 715/2119 [16:18<32:01,  1.37s/it]\u001b[A\n",
      " 34%|███▍      | 717/2119 [16:20<31:57,  1.37s/it]\u001b[A\n",
      " 34%|███▍      | 720/2119 [16:32<32:08,  1.38s/it]\u001b[A\n",
      " 34%|███▍      | 723/2119 [16:32<31:56,  1.37s/it]\u001b[A\n",
      " 34%|███▍      | 724/2119 [16:36<31:59,  1.38s/it]\u001b[A\n",
      " 34%|███▍      | 726/2119 [16:36<31:52,  1.37s/it]\u001b[A\n",
      " 34%|███▍      | 728/2119 [16:42<31:56,  1.38s/it]\u001b[A\n",
      " 34%|███▍      | 729/2119 [16:49<32:05,  1.39s/it]\u001b[A\n",
      " 34%|███▍      | 731/2119 [16:55<32:08,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 732/2119 [16:56<32:05,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 733/2119 [16:56<32:01,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 734/2119 [16:58<32:01,  1.39s/it]\u001b[A\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 35%|███▍      | 736/2119 [17:02<32:00,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 737/2119 [17:02<31:56,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 738/2119 [17:02<31:53,  1.39s/it]\u001b[A\n",
      " 35%|███▍      | 739/2119 [17:02<31:50,  1.38s/it]\u001b[A\n",
      " 35%|███▍      | 740/2119 [17:03<31:46,  1.38s/it]\u001b[A\n",
      " 35%|███▍      | 741/2119 [17:03<31:43,  1.38s/it]\u001b[A\n",
      " 35%|███▌      | 743/2119 [17:10<31:48,  1.39s/it]\u001b[A\n",
      " 35%|███▌      | 744/2119 [17:19<32:00,  1.40s/it]\u001b[A\n",
      " 35%|███▌      | 746/2119 [17:25<32:03,  1.40s/it]\u001b[A\n",
      " 35%|███▌      | 747/2119 [17:25<32:00,  1.40s/it]\u001b[A\n",
      " 35%|███▌      | 749/2119 [17:30<32:01,  1.40s/it]\u001b[A\n",
      " 35%|███▌      | 750/2119 [17:30<31:58,  1.40s/it]\u001b[A\n",
      " 35%|███▌      | 751/2119 [17:31<31:55,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 755/2119 [17:35<31:47,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 758/2119 [17:35<31:35,  1.39s/it]\u001b[A\n",
      " 36%|███▌      | 760/2119 [17:41<31:37,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 761/2119 [17:47<31:45,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 763/2119 [17:47<31:37,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 764/2119 [17:48<31:34,  1.40s/it]\u001b[A\n",
      " 36%|███▌      | 767/2119 [17:55<31:35,  1.40s/it]\u001b[A\n",
      " 36%|███▋      | 771/2119 [17:55<31:20,  1.40s/it]\u001b[A\n",
      " 36%|███▋      | 773/2119 [18:10<31:39,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 774/2119 [18:11<31:36,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 780/2119 [18:11<31:14,  1.40s/it]\u001b[A\n",
      " 37%|███▋      | 781/2119 [18:18<31:22,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 783/2119 [18:23<31:23,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 785/2119 [18:24<31:16,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 786/2119 [18:24<31:13,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 787/2119 [18:29<31:17,  1.41s/it]\u001b[A\n",
      " 37%|███▋      | 790/2119 [18:29<31:06,  1.40s/it]\u001b[A\n",
      " 37%|███▋      | 791/2119 [18:29<31:03,  1.40s/it]\u001b[A\n",
      " 37%|███▋      | 792/2119 [18:30<31:00,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 795/2119 [18:30<30:50,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 796/2119 [18:31<30:46,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 797/2119 [18:31<30:43,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 798/2119 [18:31<30:40,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 799/2119 [18:32<30:37,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 801/2119 [18:32<30:30,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 802/2119 [18:38<30:36,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 803/2119 [18:38<30:33,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 804/2119 [18:38<30:30,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 805/2119 [18:47<30:40,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 806/2119 [18:50<30:42,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 808/2119 [18:51<30:35,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 809/2119 [18:53<30:34,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 810/2119 [18:53<30:31,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 813/2119 [18:53<30:20,  1.39s/it]\u001b[A\n",
      " 38%|███▊      | 814/2119 [19:01<30:30,  1.40s/it]\u001b[A\n",
      " 38%|███▊      | 815/2119 [19:01<30:27,  1.40s/it]\u001b[A\n",
      " 39%|███▊      | 816/2119 [19:02<30:23,  1.40s/it]\u001b[A\n",
      " 39%|███▊      | 817/2119 [19:02<30:20,  1.40s/it]\u001b[A\n",
      " 39%|███▊      | 819/2119 [19:02<30:14,  1.40s/it]\u001b[A\n",
      " 39%|███▊      | 820/2119 [19:03<30:11,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 822/2119 [19:03<30:04,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 823/2119 [19:04<30:02,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 824/2119 [19:04<29:59,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 825/2119 [19:05<29:55,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 826/2119 [19:06<29:55,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 828/2119 [19:13<29:57,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 830/2119 [19:13<29:51,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 832/2119 [19:16<29:49,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 833/2119 [19:19<29:50,  1.39s/it]\u001b[A\n",
      " 39%|███▉      | 834/2119 [19:27<29:58,  1.40s/it]\u001b[A\n",
      " 39%|███▉      | 835/2119 [19:29<29:58,  1.40s/it]\u001b[A\n",
      " 39%|███▉      | 836/2119 [19:32<29:59,  1.40s/it]\u001b[A\n",
      " 39%|███▉      | 837/2119 [19:34<29:59,  1.40s/it]\u001b[A\n",
      " 40%|███▉      | 839/2119 [19:43<30:06,  1.41s/it]\u001b[A\n",
      " 40%|███▉      | 840/2119 [19:45<30:04,  1.41s/it]\u001b[A\n",
      " 40%|███▉      | 841/2119 [19:50<30:08,  1.42s/it]\u001b[A\n",
      " 40%|███▉      | 842/2119 [19:50<30:05,  1.41s/it]\u001b[A\n",
      " 40%|███▉      | 844/2119 [19:55<30:05,  1.42s/it]\u001b[A\n",
      " 40%|███▉      | 847/2119 [19:59<30:01,  1.42s/it]\u001b[A\n",
      " 40%|████      | 849/2119 [20:01<29:57,  1.42s/it]\u001b[A\n",
      " 40%|████      | 851/2119 [20:01<29:50,  1.41s/it]\u001b[A\n",
      " 40%|████      | 854/2119 [20:21<30:09,  1.43s/it]\u001b[A\n",
      " 40%|████      | 856/2119 [20:23<30:05,  1.43s/it]\u001b[A\n",
      " 41%|████      | 859/2119 [20:25<29:57,  1.43s/it]\u001b[A\n",
      " 41%|████      | 860/2119 [20:32<30:04,  1.43s/it]\u001b[A\n",
      " 41%|████      | 861/2119 [20:32<30:01,  1.43s/it]\u001b[A\n",
      " 41%|████      | 863/2119 [20:33<29:54,  1.43s/it]\u001b[A\n",
      " 41%|████      | 865/2119 [20:41<29:59,  1.43s/it]\u001b[A\n",
      " 41%|████      | 866/2119 [20:41<29:56,  1.43s/it]\u001b[A\n",
      " 41%|████      | 867/2119 [20:41<29:53,  1.43s/it]\u001b[A\n",
      " 41%|████      | 869/2119 [20:42<29:46,  1.43s/it]\u001b[A\n",
      " 41%|████      | 870/2119 [20:46<29:48,  1.43s/it]\u001b[A\n",
      " 41%|████      | 872/2119 [20:53<29:52,  1.44s/it]\u001b[A\n",
      " 41%|████      | 873/2119 [21:00<29:58,  1.44s/it]\u001b[A\n",
      " 41%|████      | 874/2119 [21:00<29:55,  1.44s/it]\u001b[A\n",
      " 41%|████▏     | 877/2119 [21:00<29:45,  1.44s/it]\u001b[A\n",
      " 41%|████▏     | 878/2119 [21:04<29:46,  1.44s/it]\u001b[A\n",
      " 41%|████▏     | 879/2119 [21:04<29:43,  1.44s/it]\u001b[A\n",
      " 42%|████▏     | 880/2119 [21:04<29:40,  1.44s/it]\u001b[A\n",
      " 42%|████▏     | 881/2119 [21:05<29:37,  1.44s/it]\u001b[A\n",
      " 42%|████▏     | 882/2119 [21:12<29:44,  1.44s/it]\u001b[A\n",
      " 42%|████▏     | 883/2119 [21:12<29:41,  1.44s/it]\u001b[A\n",
      " 42%|████▏     | 884/2119 [21:26<29:56,  1.46s/it]\u001b[A\n",
      " 42%|████▏     | 885/2119 [21:26<29:53,  1.45s/it]\u001b[A\n",
      " 42%|████▏     | 886/2119 [21:26<29:50,  1.45s/it]\u001b[A\n",
      " 42%|████▏     | 887/2119 [21:29<29:50,  1.45s/it]\u001b[A\n",
      " 42%|████▏     | 888/2119 [21:31<29:50,  1.45s/it]\u001b[A\n",
      " 42%|████▏     | 889/2119 [21:32<29:48,  1.45s/it]\u001b[A\n",
      " 42%|████▏     | 890/2119 [21:37<29:51,  1.46s/it]\u001b[A\n",
      " 42%|████▏     | 891/2119 [21:44<29:57,  1.46s/it]\u001b[A\n",
      " 42%|████▏     | 892/2119 [21:45<29:55,  1.46s/it]\u001b[A\n",
      " 42%|████▏     | 893/2119 [21:45<29:51,  1.46s/it]\u001b[A\n",
      " 42%|████▏     | 894/2119 [21:51<29:56,  1.47s/it]\u001b[A\n",
      " 42%|████▏     | 895/2119 [21:53<29:56,  1.47s/it]\u001b[A\n",
      " 42%|████▏     | 896/2119 [22:02<30:05,  1.48s/it]\u001b[A\n",
      " 42%|████▏     | 898/2119 [22:08<30:06,  1.48s/it]\u001b[A\n",
      " 42%|████▏     | 899/2119 [22:09<30:03,  1.48s/it]\u001b[A\n",
      " 42%|████▏     | 900/2119 [22:17<30:11,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 901/2119 [22:18<30:09,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 903/2119 [22:23<30:08,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 904/2119 [22:25<30:08,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 906/2119 [22:25<30:01,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 907/2119 [22:26<29:58,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 908/2119 [22:26<29:55,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 909/2119 [22:26<29:52,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 910/2119 [22:26<29:49,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 912/2119 [22:32<29:50,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 913/2119 [22:33<29:47,  1.48s/it]\u001b[A\n",
      " 43%|████▎     | 914/2119 [22:38<29:51,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 915/2119 [22:45<29:56,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 916/2119 [22:48<29:57,  1.49s/it]\u001b[A\n",
      " 43%|████▎     | 917/2119 [22:58<30:07,  1.50s/it]\u001b[A\n",
      " 43%|████▎     | 918/2119 [22:59<30:04,  1.50s/it]\u001b[A\n",
      " 43%|████▎     | 919/2119 [23:01<30:03,  1.50s/it]\u001b[A\n",
      " 43%|████▎     | 920/2119 [23:01<30:00,  1.50s/it]\u001b[A\n",
      " 43%|████▎     | 921/2119 [23:01<29:57,  1.50s/it]\u001b[A\n",
      " 44%|████▎     | 923/2119 [23:01<29:50,  1.50s/it]\u001b[A\n",
      " 44%|████▎     | 925/2119 [23:01<29:43,  1.49s/it]\u001b[A\n",
      " 44%|████▎     | 926/2119 [23:02<29:40,  1.49s/it]\u001b[A\n",
      " 44%|████▎     | 927/2119 [23:11<29:49,  1.50s/it]\u001b[A\n",
      " 44%|████▍     | 928/2119 [23:18<29:54,  1.51s/it]\u001b[A\n",
      " 44%|████▍     | 932/2119 [23:19<29:42,  1.50s/it]\u001b[A\n",
      " 44%|████▍     | 934/2119 [23:25<29:42,  1.50s/it]\u001b[A\n",
      " 44%|████▍     | 936/2119 [23:29<29:41,  1.51s/it]\u001b[A\n",
      " 44%|████▍     | 937/2119 [23:31<29:40,  1.51s/it]\u001b[A\n",
      " 44%|████▍     | 938/2119 [23:33<29:39,  1.51s/it]\u001b[A\n",
      " 44%|████▍     | 939/2119 [23:33<29:36,  1.51s/it]\u001b[A\n",
      " 44%|████▍     | 940/2119 [23:33<29:33,  1.50s/it]\u001b[A\n",
      " 44%|████▍     | 942/2119 [23:40<29:34,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 943/2119 [23:43<29:34,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 944/2119 [23:43<29:31,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 945/2119 [23:48<29:34,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 946/2119 [23:50<29:34,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 947/2119 [23:54<29:34,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 948/2119 [23:54<29:32,  1.51s/it]\u001b[A\n",
      " 45%|████▍     | 949/2119 [24:01<29:37,  1.52s/it]\u001b[A\n",
      " 45%|████▍     | 950/2119 [24:01<29:34,  1.52s/it]\u001b[A\n",
      " 45%|████▍     | 951/2119 [24:05<29:35,  1.52s/it]\u001b[A\n",
      " 45%|████▍     | 952/2119 [24:06<29:32,  1.52s/it]\u001b[A\n",
      " 45%|████▍     | 953/2119 [24:21<29:47,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 954/2119 [24:22<29:45,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 955/2119 [24:22<29:42,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 956/2119 [24:28<29:46,  1.54s/it]\u001b[A\n",
      " 45%|████▌     | 957/2119 [24:28<29:43,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 958/2119 [24:30<29:41,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 959/2119 [24:33<29:42,  1.54s/it]\u001b[A\n",
      " 45%|████▌     | 960/2119 [24:33<29:39,  1.53s/it]\u001b[A\n",
      " 45%|████▌     | 961/2119 [24:35<29:37,  1.54s/it]\u001b[A\n",
      " 45%|████▌     | 962/2119 [24:38<29:38,  1.54s/it]\u001b[A\n",
      " 45%|████▌     | 963/2119 [24:42<29:39,  1.54s/it]\u001b[A\n",
      " 45%|████▌     | 964/2119 [24:46<29:41,  1.54s/it]\u001b[A\n",
      " 46%|████▌     | 965/2119 [24:53<29:45,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 966/2119 [24:53<29:43,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 967/2119 [24:57<29:44,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 968/2119 [25:00<29:43,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 969/2119 [25:02<29:43,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 970/2119 [25:04<29:42,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 971/2119 [25:04<29:39,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 972/2119 [25:09<29:41,  1.55s/it]\u001b[A\n",
      " 46%|████▌     | 973/2119 [25:20<29:50,  1.56s/it]\u001b[A\n",
      " 46%|████▌     | 974/2119 [25:36<30:05,  1.58s/it]\u001b[A\n",
      " 46%|████▌     | 975/2119 [25:36<30:03,  1.58s/it]\u001b[A\n",
      " 46%|████▌     | 976/2119 [25:45<30:09,  1.58s/it]\u001b[A\n",
      " 46%|████▌     | 978/2119 [25:50<30:08,  1.58s/it]\u001b[A\n",
      " 46%|████▌     | 980/2119 [25:58<30:10,  1.59s/it]\u001b[A\n",
      " 46%|████▋     | 981/2119 [26:07<30:18,  1.60s/it]\u001b[A\n",
      " 46%|████▋     | 982/2119 [26:07<30:15,  1.60s/it]\u001b[A\n",
      " 46%|████▋     | 983/2119 [26:13<30:18,  1.60s/it]\u001b[A\n",
      " 46%|████▋     | 984/2119 [26:18<30:20,  1.60s/it]\u001b[A\n",
      " 46%|████▋     | 985/2119 [26:25<30:24,  1.61s/it]\u001b[A\n",
      " 47%|████▋     | 986/2119 [26:30<30:27,  1.61s/it]\u001b[A\n",
      " 47%|████▋     | 987/2119 [26:34<30:29,  1.62s/it]\u001b[A\n",
      " 47%|████▋     | 989/2119 [26:35<30:22,  1.61s/it]\u001b[A\n",
      " 47%|████▋     | 991/2119 [26:35<30:15,  1.61s/it]\u001b[A\n",
      " 47%|████▋     | 992/2119 [26:37<30:15,  1.61s/it]\u001b[A\n",
      " 47%|████▋     | 994/2119 [26:46<30:18,  1.62s/it]\u001b[A\n",
      " 47%|████▋     | 995/2119 [26:55<30:25,  1.62s/it]\u001b[A\n",
      " 47%|████▋     | 996/2119 [26:57<30:23,  1.62s/it]\u001b[A\n",
      " 47%|████▋     | 997/2119 [26:57<30:20,  1.62s/it]\u001b[A\n",
      " 47%|████▋     | 998/2119 [27:06<30:26,  1.63s/it]\u001b[A\n",
      " 47%|████▋     | 999/2119 [27:13<30:31,  1.63s/it]\u001b[A\n",
      " 47%|████▋     | 1000/2119 [27:13<30:27,  1.63s/it]\u001b[A\n",
      " 47%|████▋     | 1001/2119 [27:15<30:26,  1.63s/it]\u001b[A\n",
      " 47%|████▋     | 1002/2119 [27:23<30:31,  1.64s/it]\u001b[A\n",
      " 47%|████▋     | 1003/2119 [27:23<30:28,  1.64s/it]\u001b[A\n",
      " 47%|████▋     | 1004/2119 [27:29<30:31,  1.64s/it]\u001b[A\n",
      " 47%|████▋     | 1005/2119 [27:29<30:28,  1.64s/it]\u001b[A\n",
      " 47%|████▋     | 1006/2119 [27:38<30:35,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1008/2119 [27:39<30:28,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1009/2119 [27:43<30:30,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1011/2119 [27:44<30:23,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1012/2119 [27:50<30:27,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1013/2119 [27:58<30:32,  1.66s/it]\u001b[A\n",
      " 48%|████▊     | 1016/2119 [28:00<30:24,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1017/2119 [28:01<30:21,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1018/2119 [28:04<30:21,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1019/2119 [28:04<30:18,  1.65s/it]\u001b[A\n",
      " 48%|████▊     | 1020/2119 [28:13<30:24,  1.66s/it]\u001b[A\n",
      " 48%|████▊     | 1021/2119 [28:20<30:29,  1.67s/it]\u001b[A\n",
      " 48%|████▊     | 1022/2119 [28:27<30:32,  1.67s/it]\u001b[A\n",
      " 48%|████▊     | 1023/2119 [28:28<30:30,  1.67s/it]\u001b[A\n",
      " 48%|████▊     | 1024/2119 [28:34<30:33,  1.67s/it]\u001b[A\n",
      " 48%|████▊     | 1026/2119 [28:34<30:26,  1.67s/it]\u001b[A\n",
      " 48%|████▊     | 1027/2119 [28:34<30:23,  1.67s/it]\u001b[A\n",
      " 49%|████▊     | 1028/2119 [28:38<30:23,  1.67s/it]\u001b[A\n",
      " 49%|████▊     | 1030/2119 [28:38<30:17,  1.67s/it]\u001b[A\n",
      " 49%|████▊     | 1031/2119 [28:38<30:13,  1.67s/it]\u001b[A\n",
      " 49%|████▊     | 1032/2119 [28:38<30:10,  1.67s/it]\u001b[A\n",
      " 49%|████▊     | 1033/2119 [28:45<30:13,  1.67s/it]\u001b[A\n",
      " 49%|████▉     | 1034/2119 [28:48<30:13,  1.67s/it]\u001b[A\n",
      " 49%|████▉     | 1035/2119 [28:56<30:19,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1036/2119 [28:57<30:16,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1037/2119 [28:57<30:13,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1038/2119 [29:03<30:16,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1039/2119 [29:04<30:13,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1040/2119 [29:07<30:12,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1042/2119 [29:12<30:11,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1043/2119 [29:14<30:10,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1044/2119 [29:14<30:07,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1045/2119 [29:20<30:09,  1.68s/it]\u001b[A\n",
      " 49%|████▉     | 1046/2119 [29:26<30:11,  1.69s/it]\u001b[A\n",
      " 49%|████▉     | 1047/2119 [29:26<30:09,  1.69s/it]\u001b[A\n",
      " 49%|████▉     | 1048/2119 [29:34<30:13,  1.69s/it]\u001b[A\n",
      " 50%|████▉     | 1049/2119 [29:39<30:14,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1050/2119 [29:40<30:12,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1051/2119 [29:46<30:15,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1052/2119 [29:51<30:17,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1053/2119 [29:52<30:14,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1054/2119 [29:55<30:13,  1.70s/it]\u001b[A\n",
      " 50%|████▉     | 1055/2119 [29:59<30:14,  1.71s/it]\u001b[A\n",
      " 50%|████▉     | 1056/2119 [30:04<30:16,  1.71s/it]\u001b[A\n",
      " 50%|████▉     | 1059/2119 [30:09<30:10,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1060/2119 [30:12<30:10,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1062/2119 [30:13<30:04,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1063/2119 [30:13<30:01,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1064/2119 [30:19<30:04,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1065/2119 [30:24<30:05,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1066/2119 [30:25<30:02,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1067/2119 [30:25<29:59,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1068/2119 [30:25<29:56,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1069/2119 [30:32<29:59,  1.71s/it]\u001b[A\n",
      " 50%|█████     | 1070/2119 [30:40<30:04,  1.72s/it]\u001b[A\n",
      " 51%|█████     | 1071/2119 [30:40<30:00,  1.72s/it]\u001b[A\n",
      " 51%|█████     | 1072/2119 [30:40<29:57,  1.72s/it]\u001b[A\n",
      " 51%|█████     | 1073/2119 [30:49<30:03,  1.72s/it]\u001b[A\n",
      " 51%|█████     | 1074/2119 [31:00<30:09,  1.73s/it]\u001b[A\n",
      " 51%|█████     | 1077/2119 [31:08<30:07,  1.73s/it]\u001b[A\n",
      " 51%|█████     | 1078/2119 [31:12<30:08,  1.74s/it]\u001b[A\n",
      " 51%|█████     | 1079/2119 [31:12<30:04,  1.74s/it]\u001b[A\n",
      " 51%|█████     | 1080/2119 [31:12<30:01,  1.73s/it]\u001b[A\n",
      " 51%|█████     | 1081/2119 [31:18<30:04,  1.74s/it]\u001b[A\n",
      " 51%|█████     | 1083/2119 [31:25<30:03,  1.74s/it]\u001b[A\n",
      " 51%|█████     | 1084/2119 [31:31<30:06,  1.75s/it]\u001b[A\n",
      " 51%|█████     | 1085/2119 [31:38<30:09,  1.75s/it]\u001b[A\n",
      " 51%|█████▏    | 1086/2119 [31:39<30:06,  1.75s/it]\u001b[A\n",
      " 51%|█████▏    | 1087/2119 [31:46<30:10,  1.75s/it]\u001b[A\n",
      " 51%|█████▏    | 1088/2119 [31:46<30:06,  1.75s/it]\u001b[A\n",
      " 51%|█████▏    | 1089/2119 [31:51<30:07,  1.75s/it]\u001b[A\n",
      " 51%|█████▏    | 1090/2119 [31:59<30:12,  1.76s/it]\u001b[A\n",
      " 51%|█████▏    | 1091/2119 [32:00<30:09,  1.76s/it]\u001b[A\n",
      " 52%|█████▏    | 1092/2119 [32:08<30:13,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1093/2119 [32:09<30:10,  1.76s/it]\u001b[A\n",
      " 52%|█████▏    | 1095/2119 [32:12<30:06,  1.76s/it]\u001b[A\n",
      " 52%|█████▏    | 1096/2119 [32:12<30:04,  1.76s/it]\u001b[A\n",
      " 52%|█████▏    | 1097/2119 [32:15<30:03,  1.76s/it]\u001b[A\n",
      " 52%|█████▏    | 1098/2119 [32:23<30:07,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1099/2119 [32:24<30:04,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1100/2119 [32:32<30:08,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1101/2119 [32:32<30:05,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1102/2119 [32:32<30:01,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1103/2119 [32:32<29:58,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1104/2119 [32:35<29:58,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1105/2119 [32:41<29:59,  1.78s/it]\u001b[A\n",
      " 52%|█████▏    | 1106/2119 [32:41<29:56,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1107/2119 [32:48<29:59,  1.78s/it]\u001b[A\n",
      " 52%|█████▏    | 1108/2119 [32:48<29:56,  1.78s/it]\u001b[A\n",
      " 52%|█████▏    | 1109/2119 [32:49<29:53,  1.78s/it]\u001b[A\n",
      " 52%|█████▏    | 1110/2119 [32:49<29:50,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1111/2119 [32:49<29:47,  1.77s/it]\u001b[A\n",
      " 52%|█████▏    | 1112/2119 [32:49<29:43,  1.77s/it]\u001b[A\n",
      " 53%|█████▎    | 1113/2119 [32:50<29:41,  1.77s/it]\u001b[A\n",
      " 53%|█████▎    | 1115/2119 [32:56<29:40,  1.77s/it]\u001b[A\n",
      " 53%|█████▎    | 1116/2119 [33:02<29:41,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1117/2119 [33:08<29:43,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1118/2119 [33:08<29:40,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1119/2119 [33:08<29:37,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1120/2119 [33:17<29:41,  1.78s/it]\u001b[A\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 53%|█████▎    | 1121/2119 [33:17<29:38,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1122/2119 [33:20<29:37,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1123/2119 [33:28<29:41,  1.79s/it]\u001b[A\n",
      " 53%|█████▎    | 1124/2119 [33:28<29:38,  1.79s/it]\u001b[A\n",
      " 53%|█████▎    | 1126/2119 [33:29<29:32,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1127/2119 [33:29<29:28,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1128/2119 [33:33<29:28,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1129/2119 [33:33<29:25,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1130/2119 [33:34<29:22,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1131/2119 [33:34<29:19,  1.78s/it]\u001b[A\n",
      " 53%|█████▎    | 1132/2119 [33:45<29:26,  1.79s/it]\u001b[A\n",
      " 53%|█████▎    | 1133/2119 [33:45<29:23,  1.79s/it]\u001b[A\n",
      " 54%|█████▎    | 1134/2119 [34:04<29:35,  1.80s/it]\u001b[A\n",
      " 54%|█████▎    | 1135/2119 [34:07<29:34,  1.80s/it]\u001b[A\n",
      " 54%|█████▎    | 1136/2119 [34:08<29:32,  1.80s/it]\u001b[A\n",
      " 54%|█████▎    | 1137/2119 [34:16<29:36,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1139/2119 [34:16<29:29,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1140/2119 [34:17<29:26,  1.80s/it]\u001b[A\n",
      " 54%|█████▍    | 1141/2119 [34:17<29:23,  1.80s/it]\u001b[A\n",
      " 54%|█████▍    | 1142/2119 [34:25<29:27,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1143/2119 [34:30<29:28,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1144/2119 [34:31<29:25,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1145/2119 [34:33<29:23,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1146/2119 [34:35<29:22,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1147/2119 [34:35<29:18,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1149/2119 [34:35<29:12,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1151/2119 [34:37<29:07,  1.80s/it]\u001b[A\n",
      " 54%|█████▍    | 1152/2119 [34:38<29:05,  1.80s/it]\u001b[A\n",
      " 54%|█████▍    | 1153/2119 [34:44<29:06,  1.81s/it]\u001b[A\n",
      " 54%|█████▍    | 1154/2119 [34:44<29:03,  1.81s/it]\u001b[A\n",
      " 55%|█████▍    | 1155/2119 [34:45<29:00,  1.81s/it]\u001b[A\n",
      " 55%|█████▍    | 1156/2119 [34:49<29:00,  1.81s/it]\u001b[A\n",
      " 55%|█████▍    | 1157/2119 [35:01<29:07,  1.82s/it]\u001b[A\n",
      " 55%|█████▍    | 1158/2119 [35:07<29:08,  1.82s/it]\u001b[A\n",
      " 55%|█████▍    | 1159/2119 [35:08<29:06,  1.82s/it]\u001b[A\n",
      " 55%|█████▍    | 1160/2119 [35:10<29:04,  1.82s/it]\u001b[A\n",
      " 55%|█████▍    | 1161/2119 [35:18<29:08,  1.82s/it]\u001b[A\n",
      " 55%|█████▍    | 1162/2119 [35:21<29:06,  1.83s/it]\u001b[A\n",
      " 55%|█████▍    | 1163/2119 [35:23<29:05,  1.83s/it]\u001b[A\n",
      " 55%|█████▍    | 1164/2119 [35:28<29:06,  1.83s/it]\u001b[A\n",
      " 55%|█████▍    | 1165/2119 [35:28<29:03,  1.83s/it]\u001b[A\n",
      " 55%|█████▌    | 1167/2119 [35:32<28:59,  1.83s/it]\u001b[A\n",
      " 55%|█████▌    | 1168/2119 [35:35<28:58,  1.83s/it]\u001b[A\n",
      " 55%|█████▌    | 1169/2119 [35:40<28:59,  1.83s/it]\u001b[A\n",
      " 55%|█████▌    | 1170/2119 [35:43<28:58,  1.83s/it]\u001b[A\n",
      " 55%|█████▌    | 1171/2119 [35:52<29:02,  1.84s/it]\u001b[A\n",
      " 55%|█████▌    | 1172/2119 [36:03<29:08,  1.85s/it]\u001b[A\n",
      " 55%|█████▌    | 1173/2119 [36:17<29:15,  1.86s/it]\u001b[A\n",
      " 55%|█████▌    | 1174/2119 [36:26<29:20,  1.86s/it]\u001b[A\n",
      " 55%|█████▌    | 1176/2119 [36:27<29:13,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1179/2119 [36:27<29:03,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1181/2119 [36:29<28:59,  1.85s/it]\u001b[A\n",
      " 56%|█████▌    | 1183/2119 [36:36<28:57,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1184/2119 [36:40<28:57,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1185/2119 [36:42<28:56,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1186/2119 [36:46<28:55,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1187/2119 [36:50<28:55,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1188/2119 [36:51<28:52,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1189/2119 [36:52<28:50,  1.86s/it]\u001b[A\n",
      " 56%|█████▌    | 1191/2119 [36:52<28:43,  1.86s/it]\u001b[A\n",
      " 56%|█████▋    | 1192/2119 [36:52<28:40,  1.86s/it]\u001b[A\n",
      " 56%|█████▋    | 1194/2119 [36:52<28:34,  1.85s/it]\u001b[A\n",
      " 56%|█████▋    | 1195/2119 [36:52<28:31,  1.85s/it]\u001b[A\n",
      " 56%|█████▋    | 1196/2119 [36:53<28:27,  1.85s/it]\u001b[A\n",
      " 56%|█████▋    | 1197/2119 [36:56<28:27,  1.85s/it]\u001b[A\n",
      " 57%|█████▋    | 1198/2119 [36:57<28:24,  1.85s/it]\u001b[A\n",
      " 57%|█████▋    | 1199/2119 [37:06<28:28,  1.86s/it]\u001b[A\n",
      " 57%|█████▋    | 1200/2119 [37:07<28:25,  1.86s/it]\u001b[A\n",
      " 57%|█████▋    | 1201/2119 [37:07<28:22,  1.85s/it]\u001b[A\n",
      " 57%|█████▋    | 1203/2119 [37:07<28:16,  1.85s/it]\u001b[A\n",
      " 57%|█████▋    | 1205/2119 [37:19<28:18,  1.86s/it]\u001b[A\n",
      " 57%|█████▋    | 1206/2119 [37:29<28:23,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1207/2119 [37:40<28:27,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1208/2119 [37:40<28:24,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1209/2119 [37:44<28:24,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1210/2119 [37:44<28:21,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1211/2119 [37:45<28:18,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1213/2119 [37:45<28:12,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1214/2119 [37:51<28:13,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1216/2119 [37:57<28:10,  1.87s/it]\u001b[A\n",
      " 57%|█████▋    | 1217/2119 [37:57<28:08,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1220/2119 [38:07<28:05,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1221/2119 [38:09<28:03,  1.88s/it]\u001b[A\n",
      " 58%|█████▊    | 1222/2119 [38:09<28:00,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1223/2119 [38:12<27:59,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1224/2119 [38:12<27:56,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1225/2119 [38:13<27:53,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1227/2119 [38:13<27:47,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1228/2119 [38:16<27:46,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1229/2119 [38:21<27:46,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1230/2119 [38:21<27:43,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1231/2119 [38:26<27:43,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1233/2119 [38:26<27:37,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1234/2119 [38:33<27:39,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1235/2119 [38:33<27:36,  1.87s/it]\u001b[A\n",
      " 58%|█████▊    | 1236/2119 [38:42<27:39,  1.88s/it]\u001b[A\n",
      " 58%|█████▊    | 1237/2119 [38:42<27:36,  1.88s/it]\u001b[A\n",
      " 58%|█████▊    | 1239/2119 [38:46<27:32,  1.88s/it]\u001b[A\n",
      " 59%|█████▊    | 1240/2119 [38:56<27:36,  1.88s/it]\u001b[A\n",
      " 59%|█████▊    | 1241/2119 [38:57<27:34,  1.88s/it]\u001b[A\n",
      " 59%|█████▊    | 1242/2119 [39:03<27:34,  1.89s/it]\u001b[A\n",
      " 59%|█████▊    | 1243/2119 [39:11<27:37,  1.89s/it]\u001b[A\n",
      " 59%|█████▊    | 1244/2119 [39:15<27:37,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1245/2119 [39:16<27:34,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1246/2119 [39:22<27:35,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1247/2119 [39:26<27:34,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1248/2119 [39:26<27:31,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1249/2119 [39:26<27:28,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1250/2119 [39:26<27:25,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1251/2119 [39:26<27:22,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1252/2119 [39:29<27:21,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1254/2119 [39:32<27:16,  1.89s/it]\u001b[A\n",
      " 59%|█████▉    | 1255/2119 [39:43<27:20,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1256/2119 [39:43<27:17,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1258/2119 [39:46<27:13,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1259/2119 [39:50<27:12,  1.90s/it]\u001b[A\n",
      " 59%|█████▉    | 1260/2119 [39:57<27:14,  1.90s/it]\u001b[A\n",
      " 60%|█████▉    | 1261/2119 [40:10<27:19,  1.91s/it]\u001b[A\n",
      " 60%|█████▉    | 1262/2119 [40:10<27:16,  1.91s/it]\u001b[A\n",
      " 60%|█████▉    | 1263/2119 [40:23<27:22,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1264/2119 [40:23<27:19,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1265/2119 [40:23<27:16,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1266/2119 [40:30<27:17,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1267/2119 [40:31<27:14,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1268/2119 [40:33<27:12,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1269/2119 [40:33<27:10,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1270/2119 [40:34<27:07,  1.92s/it]\u001b[A\n",
      " 60%|█████▉    | 1271/2119 [40:34<27:04,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1272/2119 [40:35<27:02,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1273/2119 [40:40<27:01,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1274/2119 [40:41<26:59,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1275/2119 [40:43<26:57,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1276/2119 [40:48<26:57,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1277/2119 [40:52<26:57,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1278/2119 [40:55<26:56,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1279/2119 [41:00<26:56,  1.92s/it]\u001b[A\n",
      " 60%|██████    | 1280/2119 [41:04<26:55,  1.93s/it]\u001b[A\n",
      " 60%|██████    | 1281/2119 [41:07<26:54,  1.93s/it]\u001b[A\n",
      " 61%|██████    | 1282/2119 [41:13<26:54,  1.93s/it]\u001b[A\n",
      " 61%|██████    | 1283/2119 [41:22<26:57,  1.94s/it]\u001b[A\n",
      " 61%|██████    | 1284/2119 [41:24<26:55,  1.94s/it]\u001b[A\n",
      " 61%|██████    | 1285/2119 [41:40<27:03,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1286/2119 [41:50<27:06,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1287/2119 [41:54<27:05,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1289/2119 [41:54<26:58,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1290/2119 [41:54<26:55,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1291/2119 [42:02<26:57,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1292/2119 [42:06<26:57,  1.96s/it]\u001b[A\n",
      " 61%|██████    | 1293/2119 [42:06<26:53,  1.95s/it]\u001b[A\n",
      " 61%|██████    | 1295/2119 [42:17<26:54,  1.96s/it]\u001b[A\n",
      " 61%|██████    | 1296/2119 [42:17<26:51,  1.96s/it]\u001b[A\n",
      " 61%|██████    | 1297/2119 [42:19<26:49,  1.96s/it]\u001b[A\n",
      " 61%|██████▏   | 1298/2119 [42:20<26:46,  1.96s/it]\u001b[A\n",
      " 61%|██████▏   | 1300/2119 [42:20<26:40,  1.95s/it]\u001b[A\n",
      " 61%|██████▏   | 1301/2119 [42:20<26:37,  1.95s/it]\u001b[A\n",
      " 61%|██████▏   | 1302/2119 [42:25<26:37,  1.96s/it]\u001b[A\n",
      " 61%|██████▏   | 1303/2119 [42:33<26:39,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1304/2119 [42:39<26:39,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1305/2119 [42:39<26:36,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1307/2119 [42:39<26:30,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1308/2119 [42:45<26:30,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1309/2119 [42:46<26:27,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1311/2119 [42:46<26:21,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1312/2119 [42:57<26:25,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1313/2119 [42:57<26:22,  1.96s/it]\u001b[A\n",
      " 62%|██████▏   | 1314/2119 [43:05<26:24,  1.97s/it]\u001b[A\n",
      " 62%|██████▏   | 1315/2119 [43:12<26:24,  1.97s/it]\u001b[A\n",
      " 62%|██████▏   | 1316/2119 [43:15<26:23,  1.97s/it]\u001b[A\n",
      " 62%|██████▏   | 1318/2119 [43:20<26:20,  1.97s/it]\u001b[A\n",
      " 62%|██████▏   | 1320/2119 [43:25<26:17,  1.97s/it]\u001b[A\n",
      " 62%|██████▏   | 1321/2119 [43:36<26:20,  1.98s/it]\u001b[A\n",
      " 62%|██████▏   | 1322/2119 [43:42<26:21,  1.98s/it]\u001b[A\n",
      " 62%|██████▏   | 1323/2119 [43:43<26:18,  1.98s/it]\u001b[A\n",
      " 62%|██████▏   | 1324/2119 [43:43<26:15,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1325/2119 [43:46<26:13,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1326/2119 [43:46<26:10,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1327/2119 [43:51<26:10,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1328/2119 [43:56<26:10,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1329/2119 [43:57<26:07,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1330/2119 [43:57<26:04,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1331/2119 [43:57<26:01,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1332/2119 [43:59<25:59,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1333/2119 [44:00<25:56,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1334/2119 [44:01<25:54,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1335/2119 [44:01<25:51,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1336/2119 [44:01<25:48,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1337/2119 [44:05<25:47,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1338/2119 [44:12<25:48,  1.98s/it]\u001b[A\n",
      " 63%|██████▎   | 1339/2119 [44:20<25:49,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1340/2119 [44:27<25:50,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1341/2119 [44:34<25:51,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1342/2119 [44:34<25:48,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1343/2119 [44:34<25:45,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1344/2119 [44:35<25:42,  1.99s/it]\u001b[A\n",
      " 63%|██████▎   | 1345/2119 [44:43<25:44,  1.99s/it]\u001b[A\n",
      " 64%|██████▎   | 1346/2119 [44:47<25:43,  2.00s/it]\u001b[A\n",
      " 64%|██████▎   | 1347/2119 [44:51<25:42,  2.00s/it]\u001b[A\n",
      " 64%|██████▎   | 1348/2119 [44:51<25:39,  2.00s/it]\u001b[A\n",
      " 64%|██████▎   | 1349/2119 [44:56<25:38,  2.00s/it]\u001b[A\n",
      " 64%|██████▎   | 1350/2119 [44:56<25:35,  2.00s/it]\u001b[A\n",
      " 64%|██████▍   | 1351/2119 [45:05<25:38,  2.00s/it]\u001b[A\n",
      " 64%|██████▍   | 1352/2119 [45:09<25:37,  2.00s/it]\u001b[A\n",
      " 64%|██████▍   | 1353/2119 [45:18<25:39,  2.01s/it]\u001b[A\n",
      " 64%|██████▍   | 1354/2119 [45:27<25:40,  2.01s/it]\u001b[A\n",
      " 64%|██████▍   | 1355/2119 [45:34<25:42,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1356/2119 [45:43<25:43,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1357/2119 [45:43<25:40,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1358/2119 [45:44<25:37,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1359/2119 [45:44<25:34,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1361/2119 [45:49<25:31,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1362/2119 [45:51<25:29,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1363/2119 [45:53<25:27,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1364/2119 [45:53<25:24,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1365/2119 [45:58<25:23,  2.02s/it]\u001b[A\n",
      " 64%|██████▍   | 1366/2119 [46:04<25:23,  2.02s/it]\u001b[A\n",
      " 65%|██████▍   | 1367/2119 [46:05<25:21,  2.02s/it]\u001b[A\n",
      " 65%|██████▍   | 1368/2119 [46:06<25:18,  2.02s/it]\u001b[A\n",
      " 65%|██████▍   | 1369/2119 [46:10<25:17,  2.02s/it]\u001b[A\n",
      " 65%|██████▍   | 1370/2119 [46:12<25:15,  2.02s/it]\u001b[A\n",
      " 65%|██████▍   | 1371/2119 [46:18<25:15,  2.03s/it]\u001b[A\n",
      " 65%|██████▍   | 1372/2119 [46:25<25:16,  2.03s/it]\u001b[A\n",
      " 65%|██████▍   | 1373/2119 [46:25<25:13,  2.03s/it]\u001b[A\n",
      " 65%|██████▍   | 1374/2119 [46:30<25:13,  2.03s/it]\u001b[A\n",
      " 65%|██████▍   | 1375/2119 [46:36<25:12,  2.03s/it]\u001b[A\n",
      " 65%|██████▍   | 1376/2119 [46:59<25:22,  2.05s/it]\u001b[A\n",
      " 65%|██████▌   | 1378/2119 [47:06<25:20,  2.05s/it]\u001b[A\n",
      " 65%|██████▌   | 1381/2119 [47:16<25:16,  2.05s/it]\u001b[A\n",
      " 65%|██████▌   | 1382/2119 [47:16<25:12,  2.05s/it]\u001b[A\n",
      " 65%|██████▌   | 1383/2119 [47:21<25:12,  2.05s/it]\u001b[A\n",
      " 65%|██████▌   | 1384/2119 [47:26<25:11,  2.06s/it]\u001b[A\n",
      " 65%|██████▌   | 1385/2119 [47:27<25:08,  2.06s/it]\u001b[A\n",
      " 65%|██████▌   | 1386/2119 [47:30<25:07,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1388/2119 [47:30<25:01,  2.05s/it]\u001b[A\n",
      " 66%|██████▌   | 1389/2119 [47:35<25:00,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1390/2119 [47:43<25:01,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1391/2119 [47:45<24:59,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1392/2119 [47:53<25:00,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1393/2119 [47:54<24:57,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1394/2119 [47:58<24:57,  2.07s/it]\u001b[A\n",
      " 66%|██████▌   | 1397/2119 [48:04<24:50,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1398/2119 [48:06<24:48,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1399/2119 [48:07<24:45,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1401/2119 [48:07<24:39,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1402/2119 [48:07<24:36,  2.06s/it]\u001b[A\n",
      " 66%|██████▌   | 1403/2119 [48:08<24:33,  2.06s/it]\u001b[A\n",
      " 66%|██████▋   | 1404/2119 [48:09<24:31,  2.06s/it]\u001b[A\n",
      " 66%|██████▋   | 1405/2119 [48:14<24:30,  2.06s/it]\u001b[A\n",
      " 66%|██████▋   | 1406/2119 [48:17<24:29,  2.06s/it]\u001b[A\n",
      " 66%|██████▋   | 1407/2119 [48:22<24:29,  2.06s/it]\u001b[A\n",
      " 66%|██████▋   | 1409/2119 [48:23<24:22,  2.06s/it]\u001b[A\n",
      " 67%|██████▋   | 1410/2119 [48:23<24:19,  2.06s/it]\u001b[A\n",
      " 67%|██████▋   | 1411/2119 [48:29<24:19,  2.06s/it]\u001b[A\n",
      " 67%|██████▋   | 1412/2119 [48:30<24:17,  2.06s/it]\u001b[A\n",
      " 67%|██████▋   | 1413/2119 [48:39<24:18,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1414/2119 [48:45<24:18,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1415/2119 [48:47<24:16,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1417/2119 [48:48<24:10,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1418/2119 [48:55<24:10,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1420/2119 [48:59<24:07,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1421/2119 [48:59<24:04,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1422/2119 [49:00<24:01,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1423/2119 [49:00<23:58,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1424/2119 [49:07<23:58,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1425/2119 [49:07<23:55,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1426/2119 [49:07<23:52,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1427/2119 [49:07<23:49,  2.07s/it]\u001b[A\n",
      " 67%|██████▋   | 1428/2119 [49:08<23:46,  2.06s/it]\u001b[A\n",
      " 67%|██████▋   | 1429/2119 [49:14<23:46,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1431/2119 [49:14<23:40,  2.06s/it]\u001b[A\n",
      " 68%|██████▊   | 1432/2119 [49:17<23:38,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1433/2119 [49:20<23:37,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1434/2119 [49:33<23:40,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1435/2119 [49:33<23:37,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1436/2119 [49:42<23:38,  2.08s/it]\u001b[A\n",
      " 68%|██████▊   | 1437/2119 [49:43<23:35,  2.08s/it]\u001b[A\n",
      " 68%|██████▊   | 1438/2119 [49:43<23:32,  2.07s/it]\u001b[A\n",
      " 68%|██████▊   | 1439/2119 [49:50<23:33,  2.08s/it]\u001b[A\n",
      " 68%|██████▊   | 1440/2119 [49:56<23:32,  2.08s/it]\u001b[A\n",
      " 68%|██████▊   | 1441/2119 [49:56<23:29,  2.08s/it]\u001b[A\n",
      " 68%|██████▊   | 1442/2119 [50:14<23:35,  2.09s/it]\u001b[A\n",
      " 68%|██████▊   | 1445/2119 [50:17<23:27,  2.09s/it]\u001b[A\n",
      " 68%|██████▊   | 1446/2119 [50:31<23:30,  2.10s/it]\u001b[A\n",
      " 68%|██████▊   | 1447/2119 [50:31<23:27,  2.10s/it]\u001b[A\n",
      " 68%|██████▊   | 1448/2119 [50:34<23:26,  2.10s/it]\u001b[A\n",
      " 68%|██████▊   | 1449/2119 [50:34<23:23,  2.09s/it]\u001b[A\n",
      " 68%|██████▊   | 1450/2119 [50:35<23:20,  2.09s/it]\u001b[A\n",
      " 68%|██████▊   | 1451/2119 [50:43<23:20,  2.10s/it]\u001b[A\n",
      " 69%|██████▊   | 1452/2119 [50:43<23:18,  2.10s/it]\u001b[A\n",
      " 69%|██████▊   | 1453/2119 [50:43<23:15,  2.09s/it]\u001b[A\n",
      " 69%|██████▊   | 1454/2119 [50:45<23:13,  2.09s/it]\u001b[A\n",
      " 69%|██████▊   | 1455/2119 [50:46<23:10,  2.09s/it]\u001b[A\n",
      " 69%|██████▊   | 1456/2119 [50:54<23:10,  2.10s/it]\u001b[A\n",
      " 69%|██████▉   | 1457/2119 [51:02<23:11,  2.10s/it]\u001b[A\n",
      " 69%|██████▉   | 1458/2119 [51:02<23:08,  2.10s/it]\u001b[A\n",
      " 69%|██████▉   | 1459/2119 [51:09<23:08,  2.10s/it]\u001b[A\n",
      " 69%|██████▉   | 1460/2119 [51:13<23:07,  2.11s/it]\u001b[A\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 69%|██████▉   | 1461/2119 [51:16<23:05,  2.11s/it]\u001b[A\n",
      " 69%|██████▉   | 1462/2119 [51:17<23:02,  2.10s/it]\u001b[A\n",
      " 69%|██████▉   | 1463/2119 [51:28<23:04,  2.11s/it]\u001b[A\n",
      " 69%|██████▉   | 1464/2119 [51:40<23:07,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1465/2119 [51:40<23:04,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1466/2119 [51:40<23:01,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1467/2119 [51:46<23:00,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1468/2119 [51:48<22:58,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1469/2119 [51:49<22:55,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1470/2119 [51:59<22:57,  2.12s/it]\u001b[A\n",
      " 69%|██████▉   | 1472/2119 [52:00<22:51,  2.12s/it]\u001b[A\n",
      " 70%|██████▉   | 1473/2119 [52:00<22:48,  2.12s/it]\u001b[A\n",
      " 70%|██████▉   | 1474/2119 [52:14<22:51,  2.13s/it]\u001b[A\n",
      " 70%|██████▉   | 1475/2119 [52:14<22:48,  2.13s/it]\u001b[A\n",
      " 70%|██████▉   | 1478/2119 [52:15<22:39,  2.12s/it]\u001b[A\n",
      " 70%|██████▉   | 1479/2119 [52:25<22:41,  2.13s/it]\u001b[A\n",
      " 70%|██████▉   | 1483/2119 [52:26<22:29,  2.12s/it]\u001b[A\n",
      " 70%|███████   | 1484/2119 [52:34<22:29,  2.13s/it]\u001b[A\n",
      " 70%|███████   | 1485/2119 [52:42<22:30,  2.13s/it]\u001b[A\n",
      " 70%|███████   | 1486/2119 [52:57<22:33,  2.14s/it]\u001b[A\n",
      " 70%|███████   | 1489/2119 [52:58<22:24,  2.13s/it]\u001b[A\n",
      " 70%|███████   | 1490/2119 [52:59<22:22,  2.13s/it]\u001b[A\n",
      " 70%|███████   | 1491/2119 [53:14<22:25,  2.14s/it]\u001b[A\n",
      " 70%|███████   | 1492/2119 [53:14<22:22,  2.14s/it]\u001b[A\n",
      " 70%|███████   | 1493/2119 [53:14<22:19,  2.14s/it]\u001b[A\n",
      " 71%|███████   | 1494/2119 [53:20<22:19,  2.14s/it]\u001b[A\n",
      " 71%|███████   | 1495/2119 [53:29<22:19,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1496/2119 [53:30<22:16,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1498/2119 [53:30<22:10,  2.14s/it]\u001b[A\n",
      " 71%|███████   | 1499/2119 [53:37<22:10,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1500/2119 [53:38<22:07,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1501/2119 [53:38<22:05,  2.14s/it]\u001b[A\n",
      " 71%|███████   | 1502/2119 [53:43<22:04,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1503/2119 [54:00<22:07,  2.16s/it]\u001b[A\n",
      " 71%|███████   | 1504/2119 [54:05<22:06,  2.16s/it]\u001b[A\n",
      " 71%|███████   | 1506/2119 [54:05<22:01,  2.16s/it]\u001b[A\n",
      " 71%|███████   | 1508/2119 [54:06<21:55,  2.15s/it]\u001b[A\n",
      " 71%|███████   | 1509/2119 [54:06<21:52,  2.15s/it]\u001b[A\n",
      " 71%|███████▏  | 1510/2119 [54:06<21:49,  2.15s/it]\u001b[A\n",
      " 71%|███████▏  | 1511/2119 [54:07<21:46,  2.15s/it]\u001b[A\n",
      " 71%|███████▏  | 1512/2119 [54:11<21:45,  2.15s/it]\u001b[A\n",
      " 71%|███████▏  | 1513/2119 [54:17<21:44,  2.15s/it]\u001b[A\n",
      " 71%|███████▏  | 1514/2119 [54:18<21:42,  2.15s/it]\u001b[A\n",
      " 72%|███████▏  | 1516/2119 [54:19<21:36,  2.15s/it]\u001b[A\n",
      " 72%|███████▏  | 1517/2119 [54:28<21:37,  2.15s/it]\u001b[A\n",
      " 72%|███████▏  | 1519/2119 [54:32<21:32,  2.15s/it]\u001b[A\n",
      " 72%|███████▏  | 1520/2119 [54:42<21:33,  2.16s/it]\u001b[A\n",
      " 72%|███████▏  | 1521/2119 [54:46<21:32,  2.16s/it]\u001b[A\n",
      " 72%|███████▏  | 1522/2119 [54:46<21:29,  2.16s/it]\u001b[A\n",
      " 72%|███████▏  | 1523/2119 [55:02<21:32,  2.17s/it]\u001b[A\n",
      " 72%|███████▏  | 1524/2119 [55:16<21:34,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1525/2119 [55:22<21:34,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1526/2119 [55:28<21:33,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1527/2119 [55:29<21:30,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1529/2119 [55:35<21:27,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1530/2119 [55:35<21:24,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1531/2119 [55:36<21:21,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1532/2119 [55:36<21:18,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1534/2119 [55:38<21:13,  2.18s/it]\u001b[A\n",
      " 72%|███████▏  | 1535/2119 [55:50<21:14,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1537/2119 [55:51<21:08,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1538/2119 [55:51<21:06,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1539/2119 [55:52<21:03,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1540/2119 [55:53<21:00,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1541/2119 [55:53<20:57,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1542/2119 [55:54<20:55,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1544/2119 [55:55<20:49,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1545/2119 [55:55<20:46,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1546/2119 [55:56<20:43,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1547/2119 [56:03<20:43,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1549/2119 [56:07<20:39,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1550/2119 [56:08<20:36,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1551/2119 [56:08<20:33,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1552/2119 [56:11<20:31,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1553/2119 [56:20<20:31,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1554/2119 [56:20<20:29,  2.18s/it]\u001b[A\n",
      " 73%|███████▎  | 1555/2119 [56:21<20:26,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1556/2119 [56:21<20:23,  2.17s/it]\u001b[A\n",
      " 73%|███████▎  | 1557/2119 [56:35<20:25,  2.18s/it]\u001b[A\n",
      " 74%|███████▎  | 1558/2119 [56:35<20:22,  2.18s/it]\u001b[A\n",
      " 74%|███████▎  | 1559/2119 [56:48<20:24,  2.19s/it]\u001b[A\n",
      " 74%|███████▎  | 1562/2119 [56:48<20:15,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1565/2119 [56:56<20:09,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1566/2119 [56:56<20:06,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1568/2119 [56:57<20:00,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1569/2119 [57:07<20:01,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1570/2119 [57:09<19:59,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1571/2119 [57:13<19:57,  2.19s/it]\u001b[A\n",
      " 74%|███████▍  | 1572/2119 [57:13<19:54,  2.18s/it]\u001b[A\n",
      " 74%|███████▍  | 1573/2119 [57:24<19:55,  2.19s/it]\u001b[A\n",
      " 74%|███████▍  | 1574/2119 [57:25<19:52,  2.19s/it]\u001b[A\n",
      " 74%|███████▍  | 1575/2119 [57:39<19:54,  2.20s/it]\u001b[A\n",
      " 74%|███████▍  | 1578/2119 [57:51<19:50,  2.20s/it]\u001b[A\n",
      " 75%|███████▍  | 1582/2119 [57:51<19:38,  2.19s/it]\u001b[A\n",
      " 75%|███████▍  | 1584/2119 [58:02<19:36,  2.20s/it]\u001b[A\n",
      " 75%|███████▍  | 1586/2119 [58:02<19:30,  2.20s/it]\u001b[A\n",
      " 75%|███████▍  | 1588/2119 [58:13<19:28,  2.20s/it]\u001b[A\n",
      " 75%|███████▌  | 1590/2119 [58:14<19:22,  2.20s/it]\u001b[A\n",
      " 75%|███████▌  | 1592/2119 [58:22<19:19,  2.20s/it]\u001b[A\n",
      " 75%|███████▌  | 1593/2119 [58:22<19:16,  2.20s/it]\u001b[A\n",
      " 75%|███████▌  | 1594/2119 [58:28<19:15,  2.20s/it]\u001b[A\n",
      " 75%|███████▌  | 1595/2119 [58:40<19:16,  2.21s/it]\u001b[A\n",
      " 75%|███████▌  | 1596/2119 [58:46<19:15,  2.21s/it]\u001b[A\n",
      " 75%|███████▌  | 1597/2119 [58:47<19:12,  2.21s/it]\u001b[A\n",
      " 75%|███████▌  | 1598/2119 [58:51<19:11,  2.21s/it]\u001b[A\n",
      " 75%|███████▌  | 1599/2119 [58:57<19:10,  2.21s/it]\u001b[A\n",
      " 76%|███████▌  | 1600/2119 [59:09<19:11,  2.22s/it]\u001b[A\n",
      " 76%|███████▌  | 1601/2119 [59:14<19:09,  2.22s/it]\u001b[A\n",
      " 76%|███████▌  | 1603/2119 [59:19<19:05,  2.22s/it]\u001b[A\n",
      " 76%|███████▌  | 1604/2119 [59:26<19:05,  2.22s/it]\u001b[A\n",
      " 76%|███████▌  | 1605/2119 [59:31<19:03,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1606/2119 [59:31<19:00,  2.22s/it]\u001b[A\n",
      " 76%|███████▌  | 1607/2119 [59:36<18:59,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1608/2119 [59:45<18:59,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1609/2119 [59:46<18:56,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1610/2119 [59:50<18:55,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1611/2119 [59:59<18:54,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1612/2119 [1:00:00<18:52,  2.23s/it]\u001b[A\n",
      " 76%|███████▌  | 1613/2119 [1:00:05<18:51,  2.24s/it]\u001b[A\n",
      " 76%|███████▌  | 1614/2119 [1:00:09<18:49,  2.24s/it]\u001b[A\n",
      " 76%|███████▌  | 1615/2119 [1:00:16<18:48,  2.24s/it]\u001b[A\n",
      " 76%|███████▋  | 1616/2119 [1:00:27<18:49,  2.25s/it]\u001b[A\n",
      " 76%|███████▋  | 1618/2119 [1:00:36<18:45,  2.25s/it]\u001b[A\n",
      " 76%|███████▋  | 1620/2119 [1:00:36<18:40,  2.24s/it]\u001b[A\n",
      " 76%|███████▋  | 1621/2119 [1:00:36<18:37,  2.24s/it]\u001b[A\n",
      " 77%|███████▋  | 1622/2119 [1:00:37<18:34,  2.24s/it]\u001b[A\n",
      " 77%|███████▋  | 1623/2119 [1:00:37<18:31,  2.24s/it]\u001b[A\n",
      " 77%|███████▋  | 1624/2119 [1:00:47<18:31,  2.25s/it]\u001b[A\n",
      " 77%|███████▋  | 1625/2119 [1:00:57<18:31,  2.25s/it]\u001b[A\n",
      " 77%|███████▋  | 1626/2119 [1:00:57<18:28,  2.25s/it]\u001b[A\n",
      " 77%|███████▋  | 1627/2119 [1:00:57<18:26,  2.25s/it]\u001b[A\n",
      " 77%|███████▋  | 1628/2119 [1:01:02<18:24,  2.25s/it]\u001b[A\n",
      " 77%|███████▋  | 1630/2119 [1:01:16<18:22,  2.26s/it]\u001b[A\n",
      " 77%|███████▋  | 1631/2119 [1:01:23<18:22,  2.26s/it]\u001b[A\n",
      " 77%|███████▋  | 1632/2119 [1:01:23<18:19,  2.26s/it]\u001b[A\n",
      " 77%|███████▋  | 1633/2119 [1:01:31<18:18,  2.26s/it]\u001b[A\n",
      " 77%|███████▋  | 1634/2119 [1:01:42<18:18,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1635/2119 [1:01:53<18:19,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1637/2119 [1:01:58<18:14,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1638/2119 [1:01:59<18:12,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1639/2119 [1:01:59<18:09,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1640/2119 [1:01:59<18:06,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1641/2119 [1:02:02<18:04,  2.27s/it]\u001b[A\n",
      " 77%|███████▋  | 1642/2119 [1:02:08<18:03,  2.27s/it]\u001b[A\n",
      " 78%|███████▊  | 1643/2119 [1:02:14<18:02,  2.27s/it]\u001b[A\n",
      " 78%|███████▊  | 1644/2119 [1:02:19<18:00,  2.27s/it]\u001b[A\n",
      " 78%|███████▊  | 1645/2119 [1:02:29<18:00,  2.28s/it]\u001b[A\n",
      " 78%|███████▊  | 1646/2119 [1:02:29<17:57,  2.28s/it]\u001b[A\n",
      " 78%|███████▊  | 1647/2119 [1:02:29<17:54,  2.28s/it]\u001b[A\n",
      " 78%|███████▊  | 1648/2119 [1:02:34<17:53,  2.28s/it]\u001b[A\n",
      " 78%|███████▊  | 1649/2119 [1:02:37<17:51,  2.28s/it]\u001b[A\n",
      " 78%|███████▊  | 1650/2119 [1:02:58<17:53,  2.29s/it]\u001b[A\n",
      " 78%|███████▊  | 1652/2119 [1:03:15<17:53,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1653/2119 [1:03:27<17:53,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1655/2119 [1:03:28<17:47,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1656/2119 [1:03:28<17:44,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1658/2119 [1:03:28<17:39,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1659/2119 [1:03:39<17:39,  2.30s/it]\u001b[A\n",
      " 78%|███████▊  | 1660/2119 [1:03:50<17:39,  2.31s/it]\u001b[A\n",
      " 78%|███████▊  | 1661/2119 [1:03:50<17:36,  2.31s/it]\u001b[A\n",
      " 78%|███████▊  | 1662/2119 [1:03:50<17:33,  2.31s/it]\u001b[A\n",
      " 78%|███████▊  | 1663/2119 [1:03:51<17:30,  2.30s/it]\u001b[A\n",
      " 79%|███████▊  | 1665/2119 [1:04:04<17:28,  2.31s/it]\u001b[A\n",
      " 79%|███████▊  | 1666/2119 [1:04:04<17:25,  2.31s/it]\u001b[A\n",
      " 79%|███████▊  | 1668/2119 [1:04:18<17:23,  2.31s/it]\u001b[A\n",
      " 79%|███████▉  | 1669/2119 [1:04:19<17:20,  2.31s/it]\u001b[A\n",
      " 79%|███████▉  | 1671/2119 [1:04:28<17:17,  2.31s/it]\u001b[A\n",
      " 79%|███████▉  | 1672/2119 [1:04:31<17:15,  2.32s/it]\u001b[A\n",
      " 79%|███████▉  | 1673/2119 [1:04:43<17:15,  2.32s/it]\u001b[A\n",
      " 79%|███████▉  | 1674/2119 [1:04:53<17:15,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1675/2119 [1:04:53<17:12,  2.32s/it]\u001b[A\n",
      " 79%|███████▉  | 1676/2119 [1:04:57<17:10,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1677/2119 [1:05:05<17:09,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1678/2119 [1:05:05<17:06,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1679/2119 [1:05:05<17:03,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1680/2119 [1:05:07<17:01,  2.33s/it]\u001b[A\n",
      " 79%|███████▉  | 1682/2119 [1:05:08<16:55,  2.32s/it]\u001b[A\n",
      " 79%|███████▉  | 1683/2119 [1:05:08<16:52,  2.32s/it]\u001b[A\n",
      " 79%|███████▉  | 1684/2119 [1:05:16<16:51,  2.33s/it]\u001b[A\n",
      " 80%|███████▉  | 1687/2119 [1:05:25<16:45,  2.33s/it]\u001b[A\n",
      " 80%|███████▉  | 1688/2119 [1:05:36<16:45,  2.33s/it]\u001b[A\n",
      " 80%|███████▉  | 1689/2119 [1:05:46<16:44,  2.34s/it]\u001b[A\n",
      " 80%|███████▉  | 1690/2119 [1:05:46<16:41,  2.34s/it]\u001b[A\n",
      " 80%|███████▉  | 1691/2119 [1:05:57<16:41,  2.34s/it]\u001b[A\n",
      " 80%|███████▉  | 1692/2119 [1:06:05<16:40,  2.34s/it]\u001b[A\n",
      " 80%|███████▉  | 1694/2119 [1:06:05<16:34,  2.34s/it]\u001b[A\n",
      " 80%|████████  | 1696/2119 [1:06:06<16:29,  2.34s/it]\u001b[A\n",
      " 80%|████████  | 1697/2119 [1:06:11<16:27,  2.34s/it]\u001b[A\n",
      " 80%|████████  | 1698/2119 [1:06:22<16:27,  2.35s/it]\u001b[A\n",
      " 80%|████████  | 1699/2119 [1:06:30<16:26,  2.35s/it]\u001b[A\n",
      " 80%|████████  | 1700/2119 [1:06:31<16:23,  2.35s/it]\u001b[A\n",
      " 80%|████████  | 1701/2119 [1:06:43<16:23,  2.35s/it]\u001b[A\n",
      " 80%|████████  | 1702/2119 [1:06:55<16:23,  2.36s/it]\u001b[A\n",
      " 80%|████████  | 1703/2119 [1:06:59<16:21,  2.36s/it]\u001b[A\n",
      " 80%|████████  | 1704/2119 [1:07:07<16:20,  2.36s/it]\u001b[A\n",
      " 80%|████████  | 1705/2119 [1:07:08<16:18,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1706/2119 [1:07:10<16:15,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1707/2119 [1:07:11<16:12,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1708/2119 [1:07:18<16:11,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1709/2119 [1:07:18<16:08,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1711/2119 [1:07:19<16:03,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1712/2119 [1:07:22<16:01,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1713/2119 [1:07:24<15:58,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1715/2119 [1:07:28<15:53,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1716/2119 [1:07:29<15:51,  2.36s/it]\u001b[A\n",
      " 81%|████████  | 1717/2119 [1:07:42<15:51,  2.37s/it]\u001b[A\n",
      " 81%|████████  | 1718/2119 [1:07:51<15:50,  2.37s/it]\u001b[A\n",
      " 81%|████████  | 1719/2119 [1:07:54<15:48,  2.37s/it]\u001b[A\n",
      " 81%|████████  | 1721/2119 [1:08:02<15:44,  2.37s/it]\u001b[A\n",
      " 81%|████████▏ | 1722/2119 [1:08:22<15:45,  2.38s/it]\u001b[A\n",
      " 81%|████████▏ | 1723/2119 [1:08:29<15:44,  2.39s/it]\u001b[A\n",
      " 81%|████████▏ | 1724/2119 [1:08:30<15:41,  2.38s/it]\u001b[A\n",
      " 81%|████████▏ | 1725/2119 [1:08:42<15:41,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1727/2119 [1:08:45<15:36,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1728/2119 [1:08:49<15:34,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1729/2119 [1:08:56<15:32,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1730/2119 [1:08:59<15:30,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1731/2119 [1:09:01<15:28,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1732/2119 [1:09:01<15:25,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1734/2119 [1:09:01<15:19,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1735/2119 [1:09:02<15:16,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1736/2119 [1:09:07<15:14,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1737/2119 [1:09:10<15:12,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1738/2119 [1:09:20<15:12,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1739/2119 [1:09:25<15:10,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1740/2119 [1:09:26<15:07,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1741/2119 [1:09:26<15:04,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1742/2119 [1:09:34<15:03,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1743/2119 [1:09:34<15:00,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1744/2119 [1:09:38<14:58,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1745/2119 [1:09:41<14:56,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1746/2119 [1:09:41<14:53,  2.39s/it]\u001b[A\n",
      " 82%|████████▏ | 1747/2119 [1:09:46<14:51,  2.40s/it]\u001b[A\n",
      " 82%|████████▏ | 1748/2119 [1:09:50<14:49,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1749/2119 [1:09:54<14:47,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1750/2119 [1:10:00<14:45,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1752/2119 [1:10:00<14:39,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1753/2119 [1:10:06<14:38,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1754/2119 [1:10:06<14:35,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1755/2119 [1:10:14<14:34,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1757/2119 [1:10:24<14:30,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1759/2119 [1:10:25<14:24,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1761/2119 [1:10:32<14:20,  2.40s/it]\u001b[A\n",
      " 83%|████████▎ | 1762/2119 [1:10:45<14:20,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1763/2119 [1:10:47<14:17,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1765/2119 [1:10:54<14:13,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1766/2119 [1:11:00<14:11,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1767/2119 [1:11:01<14:08,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1768/2119 [1:11:07<14:07,  2.41s/it]\u001b[A\n",
      " 83%|████████▎ | 1769/2119 [1:11:08<14:04,  2.41s/it]\u001b[A\n",
      " 84%|████████▎ | 1770/2119 [1:11:15<14:03,  2.42s/it]\u001b[A\n",
      " 84%|████████▎ | 1771/2119 [1:11:21<14:01,  2.42s/it]\u001b[A\n",
      " 84%|████████▎ | 1772/2119 [1:11:21<13:58,  2.42s/it]\u001b[A\n",
      " 84%|████████▎ | 1773/2119 [1:11:21<13:55,  2.42s/it]\u001b[A\n",
      " 84%|████████▎ | 1774/2119 [1:11:22<13:52,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1775/2119 [1:11:24<13:50,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1776/2119 [1:11:24<13:47,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1777/2119 [1:11:36<13:46,  2.42s/it]\u001b[A\n",
      " 84%|████████▍ | 1778/2119 [1:11:40<13:44,  2.42s/it]\u001b[A\n",
      " 84%|████████▍ | 1780/2119 [1:11:40<13:38,  2.42s/it]\u001b[A\n",
      " 84%|████████▍ | 1782/2119 [1:11:40<13:33,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1783/2119 [1:11:41<13:30,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1785/2119 [1:11:44<13:25,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1787/2119 [1:11:44<13:19,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1788/2119 [1:11:51<13:18,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1789/2119 [1:11:57<13:16,  2.41s/it]\u001b[A\n",
      " 84%|████████▍ | 1790/2119 [1:12:05<13:15,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1791/2119 [1:12:07<13:12,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1792/2119 [1:12:10<13:10,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1793/2119 [1:12:14<13:08,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1794/2119 [1:12:20<13:06,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1795/2119 [1:12:24<13:04,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1796/2119 [1:12:28<13:01,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1797/2119 [1:12:28<12:59,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1798/2119 [1:12:35<12:57,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1799/2119 [1:12:35<12:54,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1800/2119 [1:12:41<12:52,  2.42s/it]\u001b[A\n",
      " 85%|████████▍ | 1801/2119 [1:12:46<12:50,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1802/2119 [1:12:46<12:48,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1803/2119 [1:12:47<12:45,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1804/2119 [1:12:50<12:43,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1805/2119 [1:12:56<12:41,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1806/2119 [1:12:56<12:38,  2.42s/it]\u001b[A\n",
      " 85%|████████▌ | 1807/2119 [1:13:03<12:36,  2.43s/it]\u001b[A\n",
      " 85%|████████▌ | 1809/2119 [1:13:09<12:32,  2.43s/it]\u001b[A\n",
      " 85%|████████▌ | 1810/2119 [1:13:17<12:30,  2.43s/it]\u001b[A\n",
      " 86%|████████▌ | 1812/2119 [1:13:24<12:26,  2.43s/it]\u001b[A\n",
      " 86%|████████▌ | 1813/2119 [1:13:34<12:25,  2.44s/it]\u001b[A\n",
      " 86%|████████▌ | 1814/2119 [1:13:48<12:24,  2.44s/it]\u001b[A\n",
      " 86%|████████▌ | 1816/2119 [1:14:02<12:21,  2.45s/it]\u001b[A\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 86%|████████▌ | 1817/2119 [1:14:12<12:20,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1819/2119 [1:14:15<12:14,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1820/2119 [1:14:18<12:12,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1821/2119 [1:14:21<12:10,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1822/2119 [1:14:24<12:07,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1825/2119 [1:14:25<11:59,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1826/2119 [1:14:33<11:57,  2.45s/it]\u001b[A\n",
      " 86%|████████▌ | 1827/2119 [1:14:37<11:55,  2.45s/it]\u001b[A\n",
      " 86%|████████▋ | 1828/2119 [1:14:37<11:52,  2.45s/it]\u001b[A\n",
      " 86%|████████▋ | 1829/2119 [1:14:37<11:49,  2.45s/it]\u001b[A\n",
      " 86%|████████▋ | 1830/2119 [1:14:48<11:48,  2.45s/it]\u001b[A\n",
      " 86%|████████▋ | 1831/2119 [1:14:58<11:47,  2.46s/it]\u001b[A\n",
      " 86%|████████▋ | 1832/2119 [1:15:06<11:45,  2.46s/it]\u001b[A\n",
      " 87%|████████▋ | 1833/2119 [1:15:16<11:44,  2.46s/it]\u001b[A\n",
      " 87%|████████▋ | 1834/2119 [1:15:25<11:43,  2.47s/it]\u001b[A\n",
      " 87%|████████▋ | 1835/2119 [1:15:29<11:41,  2.47s/it]\u001b[A\n",
      " 87%|████████▋ | 1836/2119 [1:15:34<11:38,  2.47s/it]\u001b[A\n",
      " 87%|████████▋ | 1837/2119 [1:15:34<11:36,  2.47s/it]\u001b[A\n",
      " 87%|████████▋ | 1838/2119 [1:15:46<11:35,  2.47s/it]\u001b[A\n",
      " 87%|████████▋ | 1839/2119 [1:15:54<11:33,  2.48s/it]\u001b[A\n",
      " 87%|████████▋ | 1841/2119 [1:16:00<11:28,  2.48s/it]\u001b[A\n",
      " 87%|████████▋ | 1842/2119 [1:16:05<11:26,  2.48s/it]\u001b[A\n",
      " 87%|████████▋ | 1843/2119 [1:16:16<11:25,  2.48s/it]\u001b[A\n",
      " 87%|████████▋ | 1844/2119 [1:16:27<11:24,  2.49s/it]\u001b[A\n",
      " 87%|████████▋ | 1845/2119 [1:16:27<11:21,  2.49s/it]\u001b[A\n",
      " 87%|████████▋ | 1846/2119 [1:16:32<11:19,  2.49s/it]\u001b[A\n",
      " 87%|████████▋ | 1847/2119 [1:16:45<11:18,  2.49s/it]\u001b[A\n",
      " 87%|████████▋ | 1848/2119 [1:16:51<11:16,  2.50s/it]\u001b[A\n",
      " 87%|████████▋ | 1849/2119 [1:16:52<11:13,  2.49s/it]\u001b[A\n",
      " 87%|████████▋ | 1850/2119 [1:17:04<11:12,  2.50s/it]\u001b[A\n",
      " 87%|████████▋ | 1851/2119 [1:17:11<11:10,  2.50s/it]\u001b[A\n",
      " 87%|████████▋ | 1852/2119 [1:17:12<11:07,  2.50s/it]\u001b[A\n",
      " 87%|████████▋ | 1853/2119 [1:17:18<11:05,  2.50s/it]\u001b[A\n",
      " 87%|████████▋ | 1854/2119 [1:17:20<11:03,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1855/2119 [1:17:20<11:00,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1857/2119 [1:17:23<10:55,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1858/2119 [1:17:27<10:52,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1859/2119 [1:17:29<10:50,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1860/2119 [1:17:39<10:48,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1861/2119 [1:17:40<10:46,  2.50s/it]\u001b[A\n",
      " 88%|████████▊ | 1862/2119 [1:17:48<10:44,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1863/2119 [1:17:55<10:42,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1864/2119 [1:18:03<10:40,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1865/2119 [1:18:11<10:38,  2.52s/it]\u001b[A\n",
      " 88%|████████▊ | 1866/2119 [1:18:11<10:36,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1867/2119 [1:18:12<10:33,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1868/2119 [1:18:15<10:30,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1869/2119 [1:18:16<10:28,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1870/2119 [1:18:17<10:25,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1871/2119 [1:18:30<10:24,  2.52s/it]\u001b[A\n",
      " 88%|████████▊ | 1872/2119 [1:18:31<10:21,  2.52s/it]\u001b[A\n",
      " 88%|████████▊ | 1874/2119 [1:18:31<10:16,  2.51s/it]\u001b[A\n",
      " 88%|████████▊ | 1875/2119 [1:18:42<10:14,  2.52s/it]\u001b[A\n",
      " 89%|████████▊ | 1876/2119 [1:18:54<10:13,  2.52s/it]\u001b[A\n",
      " 89%|████████▊ | 1878/2119 [1:18:54<10:07,  2.52s/it]\u001b[A\n",
      " 89%|████████▊ | 1880/2119 [1:19:10<10:03,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1881/2119 [1:19:21<10:02,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1882/2119 [1:19:27<10:00,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1884/2119 [1:19:32<09:55,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1886/2119 [1:19:32<09:49,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1887/2119 [1:19:40<09:47,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1888/2119 [1:19:42<09:45,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1891/2119 [1:19:42<09:36,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1892/2119 [1:19:51<09:34,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1893/2119 [1:19:58<09:32,  2.53s/it]\u001b[A\n",
      " 89%|████████▉ | 1894/2119 [1:20:04<09:30,  2.54s/it]\u001b[A\n",
      " 89%|████████▉ | 1895/2119 [1:20:05<09:27,  2.54s/it]\u001b[A\n",
      " 89%|████████▉ | 1896/2119 [1:20:11<09:25,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1897/2119 [1:20:11<09:23,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1898/2119 [1:20:17<09:20,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1899/2119 [1:20:17<09:18,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1900/2119 [1:20:27<09:16,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1901/2119 [1:20:33<09:14,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1902/2119 [1:20:34<09:11,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1903/2119 [1:20:34<09:08,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1906/2119 [1:20:43<09:01,  2.54s/it]\u001b[A\n",
      " 90%|████████▉ | 1907/2119 [1:20:54<08:59,  2.55s/it]\u001b[A\n",
      " 90%|█████████ | 1908/2119 [1:21:02<08:57,  2.55s/it]\u001b[A\n",
      " 90%|█████████ | 1910/2119 [1:21:12<08:53,  2.55s/it]\u001b[A\n",
      " 90%|█████████ | 1911/2119 [1:21:19<08:51,  2.55s/it]\u001b[A\n",
      " 90%|█████████ | 1912/2119 [1:21:28<08:49,  2.56s/it]\u001b[A\n",
      " 90%|█████████ | 1913/2119 [1:21:29<08:46,  2.56s/it]\u001b[A\n",
      " 90%|█████████ | 1914/2119 [1:21:42<08:45,  2.56s/it]\u001b[A\n",
      " 90%|█████████ | 1916/2119 [1:21:45<08:39,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1918/2119 [1:21:54<08:34,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1919/2119 [1:21:54<08:32,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1920/2119 [1:21:56<08:29,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1921/2119 [1:22:00<08:27,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1922/2119 [1:22:05<08:24,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1923/2119 [1:22:08<08:22,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1925/2119 [1:22:15<08:17,  2.56s/it]\u001b[A\n",
      " 91%|█████████ | 1927/2119 [1:22:26<08:12,  2.57s/it]\u001b[A\n",
      " 91%|█████████ | 1928/2119 [1:22:30<08:10,  2.57s/it]\u001b[A\n",
      " 91%|█████████ | 1929/2119 [1:22:41<08:08,  2.57s/it]\u001b[A\n",
      " 91%|█████████ | 1930/2119 [1:22:50<08:06,  2.58s/it]\u001b[A\n",
      " 91%|█████████ | 1931/2119 [1:23:01<08:04,  2.58s/it]\u001b[A\n",
      " 91%|█████████ | 1932/2119 [1:23:14<08:03,  2.58s/it]\u001b[A\n",
      " 91%|█████████ | 1933/2119 [1:23:19<08:01,  2.59s/it]\u001b[A\n",
      " 91%|█████████▏| 1934/2119 [1:23:20<07:58,  2.59s/it]\u001b[A\n",
      " 91%|█████████▏| 1935/2119 [1:23:20<07:55,  2.58s/it]\u001b[A\n",
      " 91%|█████████▏| 1936/2119 [1:23:27<07:53,  2.59s/it]\u001b[A\n",
      " 91%|█████████▏| 1937/2119 [1:23:29<07:50,  2.59s/it]\u001b[A\n",
      " 91%|█████████▏| 1938/2119 [1:23:38<07:48,  2.59s/it]\u001b[A\n",
      " 92%|█████████▏| 1939/2119 [1:23:38<07:45,  2.59s/it]\u001b[A\n",
      " 92%|█████████▏| 1940/2119 [1:23:38<07:43,  2.59s/it]\u001b[A\n",
      " 92%|█████████▏| 1941/2119 [1:23:49<07:41,  2.59s/it]\u001b[A\n",
      " 92%|█████████▏| 1942/2119 [1:23:49<07:38,  2.59s/it]\u001b[A\n",
      " 92%|█████████▏| 1943/2119 [1:24:02<07:36,  2.60s/it]\u001b[A\n",
      " 92%|█████████▏| 1944/2119 [1:24:14<07:35,  2.60s/it]\u001b[A\n",
      " 92%|█████████▏| 1945/2119 [1:24:17<07:32,  2.60s/it]\u001b[A\n",
      " 92%|█████████▏| 1947/2119 [1:24:21<07:27,  2.60s/it]\u001b[A\n",
      " 92%|█████████▏| 1948/2119 [1:24:22<07:24,  2.60s/it]\u001b[A\n",
      " 92%|█████████▏| 1949/2119 [1:24:36<07:22,  2.60s/it]\u001b[A"
     ]
    }
   ],
   "source": [
    "myL=[]\n",
    "for vcfDir in tqdm(vcfDirs):\n",
    "    myL.append(pd.read_csv(vcfDir,comment='#',header=None,sep='\\s+'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "metadata": {},
   "outputs": [],
   "source": [
    "meregdDf=pd.concat(myL,axis=0,keys=vcfDirs,names=['fdir'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2119"
      ]
     },
     "execution_count": 202,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(myL)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "metadata": {},
   "outputs": [],
   "source": [
    "meregdDf.to_pickle(outDir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "### analyze the allelic fraction against the primary tumors for somatic mutation calling?\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!ls -lath ./TCGA_VCF.pickle"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "asdasdasd"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!gunzip -c /nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz | head -n 23"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "scrolled": true
   },
   "source": [
    "\n",
    "!rm head.txt\n",
    "#MuSE_call_1\n",
    "\"\"\"\n",
    "with gzip.open('file.txt.gz', 'rb') as f:\n",
    "    file_content = f.read()\n",
    "\"\"\"\n",
    "fL=[]\n",
    "for i,vcfDir in enumerate(tqdm(vcfDirs)):\n",
    "    \n",
    "    with gzip.open(vcfDir, 'r') as f:\n",
    "        #.decode('ascii')\n",
    "        file_content = f.read().decode('ascii')\n",
    "\n",
    "    #str(file_content)\n",
    "    #print ( ('MuSE'  in file_content))\n",
    "    if not ('MuSE'  in file_content):\n",
    "        print (vcfDir)\n",
    "        os.system('echo \"{}\\n\" >>head.txt'.format(vcfDir))\n",
    "        os.system('gunzip -c {} | head -\n",
    "                  n 20 >>head.txt'.format(vcfDir))\n",
    "\n",
    "        os.system('echo \"\\n\" >> head.txt'.format(vcfDir))\n",
    "    fL.append(vcfDir)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 172,
   "metadata": {},
   "outputs": [],
   "source": [
    "#lgg_rest_meta_df=pd.read_pickle('lgg_rest_meta.pickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [],
   "source": [
    "#lgg_rest_meta_df.groupby('var_pipeline').first()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [],
   "source": [
    "#529 files having all \n",
    "#'varscan_dr', 'muse', 'mutect', 'somaticsniper'\n",
    "#lgg_rest_meta_df.var_pipeline.value_counts().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {},
   "outputs": [],
   "source": [
    "#lgg_rest_meta_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([], dtype=object)"
      ]
     },
     "execution_count": 167,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lgg_rest_meta_df['submitter_id'][~lgg_rest_meta_df['submitter_id'].str.contains('varscan_dr|somaticsniper|mutect|muse')].values#.str.split('_').str[-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/99690673-b98b-496c-bf5e-e08d3b3dc545/99690673-b98b-496c-bf5e-e08d3b3dc545.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job127557346_index_O09uzk/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1020217\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,2,0:33:36:0:26:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,0,0,2:33:36:33:33:60:60:2:16\r\n",
      "chr1\t1022591\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:9:0,9,0,0:0,0,9,0:54:60:0:33:52:52:0:.\t0/1:0/1:15:0,10,0,5:5,0,10,0:53:60:53:32,33:54:60,52:2:37\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c48297db-2dcd-4216-9033-f9bdbe433775/c48297db-2dcd-4216-9033-f9bdbe433775.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/dbfb00be-ed30-402f-bb2e-19e7d0ec340f/dbfb00be-ed30-402f-bb2e-19e7d0ec340f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4aa94453-1f1c-4120-864e-6b018f10704e/4aa94453-1f1c-4120-864e-6b018f10704e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/52615c7e-1274-4513-bd11-c063aef4ae0f/52615c7e-1274-4513-bd11-c063aef4ae0f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/eef4b992-9147-45d9-9530-eaf2fedb2015/eef4b992-9147-45d9-9530-eaf2fedb2015.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/7788aef6-5cae-47e7-ae87-9ebba4e9facb/7788aef6-5cae-47e7-ae87-9ebba4e9facb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2e7db8f4-043d-4ec5-ad72-1f5f60cb9d5f/2e7db8f4-043d-4ec5-ad72-1f5f60cb9d5f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/fd259142-5e82-4127-8bff-e611df8be734/fd259142-5e82-4127-8bff-e611df8be734.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/21f60c50-d463-422c-a5dd-b3b19bcaeab3/21f60c50-d463-422c-a5dd-b3b19bcaeab3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44125418-c4b3-4c2a-9019-d88493a7116b/44125418-c4b3-4c2a-9019-d88493a7116b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4d26f14f-16e1-4ec0-a8b4-fbc83957493a/4d26f14f-16e1-4ec0-a8b4-fbc83957493a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job780864691_index_B8Gkn0/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t187497\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:132:72,36,20,4:24,0,108,0:52:120:0:29:29:31:0:.\t0/1:0/1:104:45,33,16,10:26,0,78,0:113:120:113:34,30:33:29,35:2:35\r\n",
      "chr1\t908209\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:20:0:35:60:60:0:.\t0/1:0/1:8:4,2,1,1:2,0,6,0:13:20:13:32,35:60:60,60:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f89f8da-d257-460e-bde6-798804da6a43/6f89f8da-d257-460e-bde6-798804da6a43.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/59e35df7-42dc-47d0-99e5-71167e225c9f/59e35df7-42dc-47d0-99e5-71167e225c9f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a35f0b8d-b832-479c-8547-f6f18b16dc7f/a35f0b8d-b832-479c-8547-f6f18b16dc7f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/41b6375b-42a1-4d1c-afaf-5063664265d6/41b6375b-42a1-4d1c-afaf-5063664265d6.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/1f001f73-d30e-4363-9fee-509fc0c0872c/1f001f73-d30e-4363-9fee-509fc0c0872c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8ad8ebe1-4091-4486-a98f-74568b1acacb/8ad8ebe1-4091-4486-a98f-74568b1acacb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/90902bbe-e9db-4ac3-8762-c65eb343a22d/90902bbe-e9db-4ac3-8762-c65eb343a22d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job407592064_index_Coi4O2/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t873857\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:88:48,23,11,6:0,71,16,0:38:69:0:32:33:36:0:.\t0/1:0/1:142:65,37,29,11:0,102,39,1:62:69:62:31,32:30:34,21:2:21\r\n",
      "chr1\t971419\t.\tA\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:2,0,0,0:31:23:0:16:60:60:0:.\t0/1:0/1:7:1,4,1,1:5,0,0,2:16:23:16:27,32:60:60,60:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/15888361-3b65-4696-813c-bd62f1b4391b/15888361-3b65-4696-813c-bd62f1b4391b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5da2428b-21b3-4296-a3e7-a58dfdb35f89/5da2428b-21b3-4296-a3e7-a58dfdb35f89.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/513a7dd9-2f05-437e-bfd4-ab96d9ce2721/513a7dd9-2f05-437e-bfd4-ab96d9ce2721.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/1717b93f-b3bb-432f-9829-892633593aed/1717b93f-b3bb-432f-9829-892633593aed.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160310\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job399196293_index_uWI20c/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t932716\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:3,1,0,0:0,0,0,4:39:63:0:31:60:60:0:.\t0/1:0/1:5:2,0,1,2:0,3,0,2:18:63:54:34,32:60:60,60:2:22\r\n",
      "chr1\t1033565\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:36:0:31:60:60:0:.\t0/1:1/1:3:0,1,1,1:2,0,1,0:4:36:33:32:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6a8081c5-b750-4be0-a8e4-844a45646f3f/6a8081c5-b750-4be0-a8e4-844a45646f3f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/96ccc7ed-5e0b-4f7b-a92b-deab64c693e8/96ccc7ed-5e0b-4f7b-a92b-deab64c693e8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/7445e19e-fa2b-4605-ab81-32a7ab1ebf62/7445e19e-fa2b-4605-ab81-32a7ab1ebf62.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/628b61c5-50de-476b-a5a7-45c2a0d0cd89/628b61c5-50de-476b-a5a7-45c2a0d0cd89.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f55e03f4-9d54-4692-8850-da3040d6096d/f55e03f4-9d54-4692-8850-da3040d6096d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job198795809_index_BmNF8p/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t10560\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,2,0,0:33:38:0:34:30:30:0:.\t0/1:1/1:3:0,1,1,1:0,1,2,0:23:38:33:33:35:46:2:16\r\n",
      "chr1\t52096\t.\tA\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:2,0,0,0:33:24:0:33:36:36:0:.\t0/1:1/1:2:0,0,0,2:0,2,0,0:33:24:33:30:35:35:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/83ee90ad-12cf-42d4-bea7-d20924bb04fb/83ee90ad-12cf-42d4-bea7-d20924bb04fb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3fe98ee9-e85c-412c-befa-814a1eebf7ea/3fe98ee9-e85c-412c-befa-814a1eebf7ea.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4c38f26b-95c4-4dc9-9b97-1af9b04ebbcb/4c38f26b-95c4-4dc9-9b97-1af9b04ebbcb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e0db645d-7929-4b58-bee9-1edd47141ced/e0db645d-7929-4b58-bee9-1edd47141ced.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b936c5ba-97a7-4744-9bfb-88e1c3bfecf4/b936c5ba-97a7-4744-9bfb-88e1c3bfecf4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/95f3638c-1e6d-4a41-929a-c547a4716ff8/95f3638c-1e6d-4a41-929a-c547a4716ff8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job237679929_index_onB2G9/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t873857\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:90:56,22,8,4:0,78,12,0:90:17:0:32:32:33:0:.\t0/1:0/1:131:73,25,27,6:1,98,32,0:10:17:10:31,33:31:34,22:2:17\r\n",
      "chr1\t948721\t.\tA\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:17:0,17,0,0:17,0,0,0:78:41:0:30:60:60:0:.\t0/1:0/1:16:0,12,0,4:12,4,0,0:34:41:34:30,33:60:60,60:2:41\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/267961b2-e186-4bf6-a837-3b2ea2ff5b16/267961b2-e186-4bf6-a837-3b2ea2ff5b16.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/7c5e3042-559e-4ad3-bbfb-87b5a52b16c4/7c5e3042-559e-4ad3-bbfb-87b5a52b16c4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job389287182_index_i_AP3J/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t873857\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:164:80,56,19,9:0,136,28,0:86:134:0:32:27:28:0:.\t0/1:0/1:152:71,32,35,14:0,103,49,0:127:134:127:32,33:28:29,24:2:69\r\n",
      "chr1\t1182106\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:2,0,0,0:33:45:0:31:60:60:0:.\t0/1:1/1:3:0,0,3,0:0,0,3,0:36:45:36:29:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/936c32ab-ba7e-49a4-8026-0e9c5adb8f3a/936c32ab-ba7e-49a4-8026-0e9c5adb8f3a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/de2c4f4e-738f-47e2-bc66-a71f57518af4/de2c4f4e-738f-47e2-bc66-a71f57518af4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8cb7d577-57db-415e-b13a-08f689fd1b2b/8cb7d577-57db-415e-b13a-08f689fd1b2b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/844601ff-21db-4161-954b-5e1c50a4d5bc/844601ff-21db-4161-954b-5e1c50a4d5bc.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job55727519_index_NScfAF/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t966426\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:5,0,0,0:0,0,5,0:42:83:0:33:60:60:0:.\t0/1:0/1:9:4,0,5,0:0,5,4,0:58:83:76:34,35:60:60,60:2:25\r\n",
      "chr1\t1048922\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,0,2:33:38:0:34:60:60:0:.\t0/1:0/1:6:2,1,2,1:0,3,0,3:31:38:31:27,35:60:60,60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5c072661-b257-4bd4-aac5-1b44646f5e31/5c072661-b257-4bd4-aac5-1b44646f5e31.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5d98c68b-afb5-42d7-aae2-ec3e83ecf9c0/5d98c68b-afb5-42d7-aae2-ec3e83ecf9c0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d48ab309-44d0-44f0-99b2-d65b1cc89f23/d48ab309-44d0-44f0-99b2-d65b1cc89f23.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/300952f6-348f-402f-94a7-919dc99f43bd/300952f6-348f-402f-94a7-919dc99f43bd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/478b4f88-6e7c-4e76-a3f3-7824437c5ff1/478b4f88-6e7c-4e76-a3f3-7824437c5ff1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/86648923-2ea2-4825-9c93-c550d59a4d8e/86648923-2ea2-4825-9c93-c550d59a4d8e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job223417748_index_w7yvpr/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1184277\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:0,3,0,0:0,0,3,0:36:23:0:31:60:60:0:.\t0/1:0/1:4:0,2,0,2:0,2,2,0:16:23:16:29,31:60:60,60:2:18\r\n",
      "chr1\t1232848\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:3,2,0,0:0,5,0,0:42:21:0:27:60:60:0:.\t0/1:0/1:5:2,1,2,0:2,3,0,0:14:21:14:30,27:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/1eae65af-8a16-4d4d-a34c-9d8af925abf6/1eae65af-8a16-4d4d-a34c-9d8af925abf6.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e5fd1c95-e4c1-4ede-8940-cfcfce7a4e8e/e5fd1c95-e4c1-4ede-8940-cfcfce7a4e8e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job399155408_index_gMOHuw/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t182686\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:3,2,0,0:5,0,0,0:42:26:0:29:22:22:0:.\t0/1:0/1:8:3,2,3,0:5,0,3,0:19:26:19:33,29:44:48,37:2:22\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:212:85,102,11,14:24,0,187,1:203:32:0:31:52:56:0:.\t0/1:0/1:103:28,52,8,15:21,1,80,0:25:32:25:34,31:49:25,56:2:32\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5593f47b-56c8-4a27-bd31-3c749725a34f/5593f47b-56c8-4a27-bd31-3c749725a34f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c66f9910-c294-405f-828a-5f5591a5edda/c66f9910-c294-405f-828a-5f5591a5edda.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job187398651_index_J_rhlt/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t957428\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,4,0,0:39:22:0:33:60:60:0:.\t0/1:1/1:2:0,0,0,2:1,0,0,1:2:22:3:32:60:60:2:22\r\n",
      "chr1\t960326\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:41:0:33:60:60:0:.\t0/1:1/1:2:0,0,1,1:2,0,0,0:33:41:33:33:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/658e9cc6-2766-46dd-998d-d250a0626a02/658e9cc6-2766-46dd-998d-d250a0626a02.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0db3d19f-26f6-448e-818d-ff412e5812c1/0db3d19f-26f6-448e-818d-ff412e5812c1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job520016403_index_Y9RMcK/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1509162\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:28:22,3,2,1:0,0,25,3:38:61:0:28:32:32:0:.\t0/1:0/1:23:16,1,4,2:0,0,17,6:54:61:54:28,32:32:33,31:2:21\r\n",
      "chr1\t1522905\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:19:6,11,0,2:17,0,2,0:46:34:0:27:45:47:0:.\t0/1:0/1:29:4,16,1,8:20,0,9,0:27:34:27:30,26:43:51,26:2:28\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3eee1a32-6c75-4cbc-a585-4bfcdcf009a7/3eee1a32-6c75-4cbc-a585-4bfcdcf009a7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job796512207_index_O2cWNU/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183697\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:7:6,1,0,0:0,0,7,0:48:30:0:29:24:24:0:.\t0/1:0/1:12:8,0,4,0:4,0,8,0:23:30:23:32,33:24:26,23:2:27\r\n",
      "chr1\t975983\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,0,2,0:33:33:0:33:60:60:0:.\t0/1:1/1:3:1,0,1,1:2,0,1,0:9:33:33:31:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4c14381c-dafd-48d7-8350-8e06450bdae0/4c14381c-dafd-48d7-8350-8e06450bdae0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/75f89481-d732-4d74-b35c-8db8130066c2/75f89481-d732-4d74-b35c-8db8130066c2.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b03250cf-acea-4e98-b9e3-13796a3b5c12/b03250cf-acea-4e98-b9e3-13796a3b5c12.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0ba20775-e673-48fa-82bf-0caa78c666ec/0ba20775-e673-48fa-82bf-0caa78c666ec.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/9cde07f0-7fb3-459d-a6b6-a9af744696c7/9cde07f0-7fb3-459d-a6b6-a9af744696c7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6b606d7f-ad49-4e85-984e-ff191c1954e0/6b606d7f-ad49-4e85-984e-ff191c1954e0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a7b103a9-0532-4119-a027-f4acf3feb552/a7b103a9-0532-4119-a027-f4acf3feb552.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/cef7785c-6e73-4525-acb3-9f4bde134860/cef7785c-6e73-4525-acb3-9f4bde134860.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job159272406_index_NY1bjy/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t17020\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,0,2,0:33:30:0:32:23:23:0:.\t0/1:0/1:5:1,1,1,2:3,0,2,0:3:30:23:30,32:23:23,22:2:16\r\n",
      "chr1\t17495\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:6:0,6,0,0:0,6,0,0:45:25:0:34:38:38:0:.\t0/1:0/1:5:2,1,1,1:2,3,0,0:18:25:18:30,32:42:40,43:2:23\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/491a35ed-1e0b-47f8-9ef5-4e5b9d4d1e5b/491a35ed-1e0b-47f8-9ef5-4e5b9d4d1e5b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job73561579_index_ADlClh/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t782105\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:32:0:29:40:40:0:.\t0/1:0/1:5:0,3,0,2:2,3,0,0:25:32:25:36,29:43:45,41:2:24\r\n",
      "chr1\t874352\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:129:38,67,7,17:0,24,0,105:34:20:0:32:53:60:0:.\t0/1:0/1:119:33,65,8,13:0,21,0,98:13:20:13:35,32:55:29,60:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e376dee3-78ab-4d7f-b966-3022b8c3f2d9/e376dee3-78ab-4d7f-b966-3022b8c3f2d9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job641603666_index_KSV2Kg/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1519934\t.\tT\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:33:23:0:32:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,0,2,0:33:23:33:26:60:60:2:15\r\n",
      "chr1\t1519938\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:33:36:0:32:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,2,0,0:33:36:33:33:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8748b776-fbcd-4a3c-a9a1-cb1210c5b76b/8748b776-fbcd-4a3c-a9a1-cb1210c5b76b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job621196582_index_8qFTt7/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1054900\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,2,0,0:33:108:0:30:60:60:0:.\t0/1:0/1:10:1,3,2,4:0,4,0,6:43:108:88:27,31:60:60,60:2:16\r\n",
      "chr1\t1309863\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:31:0:22:60:60:0:.\t0/1:0/1:4:0,2,1,1:2,2,0,0:24:31:24:31,34:60:60,60:2:24\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/672abd81-388e-46c1-aaf1-4cd9f255886b/672abd81-388e-46c1-aaf1-4cd9f255886b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/19c3c1e7-7d6f-4785-bc6a-4a1ca4cd9467/19c3c1e7-7d6f-4785-bc6a-4a1ca4cd9467.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job241064353_index_xW2u4o/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t182686\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:12:9,2,1,0:11,0,1,0:35:20:0:34:34:33:0:.\t0/1:0/1:8:3,2,3,0:5,0,3,0:13:20:13:31,25:45:43,47:2:16\r\n",
      "chr1\t791542\t.\tC\tA,T\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/2:2/2:2:0,0,1,1:0,0,0,2:33:30:33:33:34:34:1:.\t0/1:0/1:4:2,0,1,1:2,2,0,0:1:30:17:28,29:27:29,26:2:29\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/865f6278-b0c5-42f6-93da-e755c2c1b201/865f6278-b0c5-42f6-93da-e755c2c1b201.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/fce0210e-956d-4ab2-9fa3-65e70987f92f/fce0210e-956d-4ab2-9fa3-65e70987f92f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job513825787_index_rqZz9C/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t874352\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:121:33,76,4,8:0,12,0,109:186:68:0:31:56:60:0:.\t0/1:0/1:89:16,49,14,10:1,23,0,65:61:68:61:33,32:49:19,60:2:68\r\n",
      "chr1\t1239942\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:6:4,2,0,0:0,0,6,0:45:29:0:29:60:60:0:.\t0/1:0/1:5:2,1,1,1:0,0,3,2:22:29:22:30,32:60:60,60:2:25\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/36f902f6-d75e-4a79-8436-e7ddbef56ca8/36f902f6-d75e-4a79-8436-e7ddbef56ca8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8fe4c94a-8da9-463d-becc-3afdf8db9380/8fe4c94a-8da9-463d-becc-3afdf8db9380.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job794909707_index_1u57l9/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t187497\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:168:89,57,14,8:22,0,146,0:106:55:0:30:34:35:0:.\t0/1:0/1:180:95,41,29,15:44,0,136,0:48:55:48:32,29:28:22,30:2:55\r\n",
      "chr1\t874352\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:100:33,53,2,12:0,14,0,86:93:26:0:32:55:60:0:.\t0/1:0/1:95:32,41,9,13:0,22,0,73:19:26:19:34,32:51:20,60:2:26\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e09ec1b5-ea2f-467c-8618-a55ec2366077/e09ec1b5-ea2f-467c-8618-a55ec2366077.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job533995896_index_n2NQt5/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t972251\t.\tA\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:46:26,7,1,12:33,12,1,0:34:28:0:29:60:60:0:.\t0/1:0/1:40:19,9,0,12:28,12,0,0:21:28:21:23,21:60:60,60:2:17\r\n",
      "chr1\t1041823\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:6:6,0,0,0:0,0,6,0:45:24:0:33:60:60:0:.\t0/1:0/1:9:5,1,2,1:0,3,6,0:17:24:17:25,36:56:60,53:2:23\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/551ed907-025d-44da-9671-175f805444e1/551ed907-025d-44da-9671-175f805444e1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/653008a2-2973-4b92-8e29-b8edd649070a/653008a2-2973-4b92-8e29-b8edd649070a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d8139160-ed8c-4ce2-89c8-43e14c6adf17/d8139160-ed8c-4ce2-89c8-43e14c6adf17.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6db236f3-607c-4365-8644-0db6c9ac02c9/6db236f3-607c-4365-8644-0db6c9ac02c9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f704b4d2-af43-409a-9f3f-4ddffc24cb39/f704b4d2-af43-409a-9f3f-4ddffc24cb39.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3e892792-7dca-45c0-a51c-48344984fb4b/3e892792-7dca-45c0-a51c-48344984fb4b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job895542495_index_cYXKk0/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t929839\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:2,1,0,0:0,0,3,0:36:56:0:28:60:60:0:.\t0/1:1/1:3:0,0,1,2:3,0,0,0:36:56:36:30:60:60:2:19\r\n",
      "chr1\t929903\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,2,0,0:33:56:0:30:60:60:0:.\t0/1:1/1:4:0,1,1,2:0,1,0,3:8:56:36:31:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f76e340a-ac87-4232-af4d-bff6991a2553/f76e340a-ac87-4232-af4d-bff6991a2553.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job11684446_index_TS5CZi/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:180:69,91,12,8:19,0,160,0:229:40:0:31:52:56:0:.\t0/1:0/1:110:35,50,8,17:25,0,85,0:33:40:33:31,31:48:23,56:2:40\r\n",
      "chr1\t874307\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:180:69,93,11,7:0,18,162,0:250:27:0:34:53:57:0:.\t0/1:0/1:111:35,51,8,17:0,25,86,0:20:27:20:28,33:50:23,58:2:27\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/50896ac5-d8be-4ec3-94ac-471fc8993e4c/50896ac5-d8be-4ec3-94ac-471fc8993e4c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job945451079_index_DtN3q8/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t803762\t.\tT\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,0,2:33:39:0:28:40:40:0:.\t0/1:1/1:3:0,0,2,1:3,0,0,0:36:39:36:32:23:23:2:16\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:148:54,78,10,6:15,0,132,0:185:40:0:29:51:54:0:.\t0/1:0/1:150:43,77,14,16:29,0,120,1:33:40:33:33,30:49:26,55:2:40\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d7c48f6b-25d8-4b8b-88bc-55c81e2a28e4/d7c48f6b-25d8-4b8b-88bc-55c81e2a28e4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/dfb93061-bbe2-4f27-85df-59c57cd10f89/dfb93061-bbe2-4f27-85df-59c57cd10f89.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0ea3b6ec-d3f1-45ef-b3f3-69a9e61f93fc/0ea3b6ec-d3f1-45ef-b3f3-69a9e61f93fc.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job359730471_index_yIVh8g/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1054900\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:1,2,0,0:0,3,0,0:36:33:0:29:60:60:0:.\t0/1:1/1:2:0,0,1,1:0,0,0,2:33:33:33:29:60:60:2:19\r\n",
      "chr1\t1388206\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:11:0,11,0,0:0,11,0,0:60:20:0:36:60:60:0:.\t0/1:0/1:5:0,3,0,2:0,3,0,2:13:20:13:31,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5a84908c-5aab-41aa-aca0-d1d666727417/5a84908c-5aab-41aa-aca0-d1d666727417.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bacb549a-5425-404c-995c-3faad7a20580/bacb549a-5425-404c-995c-3faad7a20580.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/ca98ebfe-d0e4-472d-9d8d-e65b5991f812/ca98ebfe-d0e4-472d-9d8d-e65b5991f812.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/7ca78c7e-aaaf-4f0b-91f3-a71281e9e750/7ca78c7e-aaaf-4f0b-91f3-a71281e9e750.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/784bf14d-2d2c-4199-a89b-0c4faa2c6e25/784bf14d-2d2c-4199-a89b-0c4faa2c6e25.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/020879e0-9ef5-40e4-89c9-409323e86d5a/020879e0-9ef5-40e4-89c9-409323e86d5a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/7f4b5061-66db-432f-8783-509f313da6f2/7f4b5061-66db-432f-8783-509f313da6f2.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/9ad50bf6-9fe0-4ad7-b26f-b6f52824ef9e/9ad50bf6-9fe0-4ad7-b26f-b6f52824ef9e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/eb19d1eb-3497-49c9-8619-245ad90737fb/eb19d1eb-3497-49c9-8619-245ad90737fb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/14c29283-a851-450b-bed3-da067a691dd0/14c29283-a851-450b-bed3-da067a691dd0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job54932884_index_ugZDi6/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14653\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:35:29,4,2,0:0,33,0,2:99:35:0:30:21:22:0:.\t0/1:0/1:24:11,5,7,1:0,16,0,8:28:35:28:32,31:22:23,22:2:35\r\n",
      "chr1\t873857\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:103:64,19,13,7:1,83,19,0:57:59:0:31:28:30:0:.\t0/1:0/1:62:28,18,11,5:0,46,16,0:52:59:52:32,35:30:33,22:2:40\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/05e477c2-9fa7-4d9a-8a98-6b7b805348f1/05e477c2-9fa7-4d9a-8a98-6b7b805348f1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job840590575_index_ei_zjq/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1237063\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,0,2,0:33:31:0:32:60:60:0:.\t0/1:1/1:3:0,1,1,1:0,0,1,2:3:31:27:30:60:60:2:16\r\n",
      "chr1\t1455337\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:2,0,0,0:33:29:0:27:60:60:0:.\t0/1:1/1:4:1,0,3,0:1,0,3,0:12:29:34:23:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0d4b0a89-7c06-48d6-9180-e69146c5b27c/0d4b0a89-7c06-48d6-9180-e69146c5b27c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c48a1c11-9d11-4d98-b1ae-0c600757746f/c48a1c11-9d11-4d98-b1ae-0c600757746f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job111296575_index_vxlJXR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1126128\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:3,0,0,0:0,0,3,0:10:22:0:2:5:5:0:.\t0/1:1/1:3:1,0,1,1:2,0,1,0:28:22:33:31:44:60:2:17\r\n",
      "chr1\t1296255\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:10:2,8,0,0:0,10,0,0:57:17:0:32:60:60:0:.\t0/1:0/1:9:0,7,1,1:2,7,0,0:10:17:10:32,32:60:60,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/ebcba6b5-6264-44cd-b8a9-386fe086d0cb/ebcba6b5-6264-44cd-b8a9-386fe086d0cb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d5bd22ef-14ec-4173-8141-f6df4426d71c/d5bd22ef-14ec-4173-8141-f6df4426d71c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job611462398_index_3UapdL/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:181:68,92,13,8:21,0,160,0:197:42:0:28:51:54:0:.\t0/1:0/1:146:44,67,14,21:34,1,111,0:35:42:35:32,27:45:18,53:2:42\r\n",
      "chr1\t874307\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:180:65,92,13,10:0,23,157,0:174:26:0:32:53:57:0:.\t0/1:0/1:138:44,59,14,21:0,35,103,0:19:26:19:27,31:46:19,56:2:26\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/628cee15-33c7-45ee-be4f-76c4e5a7233d/628cee15-33c7-45ee-be4f-76c4e5a7233d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6da7b2d9-7ee6-4f69-a565-463a9daf3208/6da7b2d9-7ee6-4f69-a565-463a9daf3208.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job567974803_index_WFGLEf/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:204:76,103,13,12:25,0,179,0:184:198:0:30:51:55:0:.\t0/1:0/1:251:56,94,49,52:101,0,150,0:191:198:191:33,30:41:21,55:2:167\r\n",
      "chr1\t874307\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:207:76,106,13,12:0,25,182,0:208:220:0:33:53:57:0:.\t0/1:0/1:249:53,92,50,54:0,102,145,2:214:220:214:30,33:42:20,58:2:191\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/72271fed-5d7d-426d-bfcd-0e83756b10c5/72271fed-5d7d-426d-bfcd-0e83756b10c5.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/50c31528-dd1a-437d-a40c-df1142ef7843/50c31528-dd1a-437d-a40c-df1142ef7843.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2e4b8567-bec1-48dd-a13e-4476421a9ed7/2e4b8567-bec1-48dd-a13e-4476421a9ed7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/45426b88-6b59-409a-a485-59eb5aed08ab/45426b88-6b59-409a-a485-59eb5aed08ab.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2bffdf56-3e6e-4720-89bb-64eb6d5527eb/2bffdf56-3e6e-4720-89bb-64eb6d5527eb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job613377736_index_dTt4i0/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1041370\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:6:5,1,0,0:0,0,6,0:45:34:0:34:60:60:0:.\t0/1:0/1:4:2,0,1,1:0,0,2,2:25:34:27:34,33:60:60,60:2:27\r\n",
      "chr1\t1213643\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:9:6,3,0,0:0,0,9,0:54:26:0:32:60:60:0:.\t0/1:0/1:7:5,0,1,1:0,0,5,2:19:26:19:27,33:60:60,60:2:26\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2ff8f3ab-cd0e-4409-bbfa-7199caf198d5/2ff8f3ab-cd0e-4409-bbfa-7199caf198d5.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e8ace314-76d3-4089-aa4e-12a497651b34/e8ace314-76d3-4089-aa4e-12a497651b34.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/fe28e2fe-eada-461a-ad82-a3b3038b21b9/fe28e2fe-eada-461a-ad82-a3b3038b21b9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0ec42270-dfde-4561-88db-854874c30030/0ec42270-dfde-4561-88db-854874c30030.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job450580921_index_8BSXHa/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14653\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:30:20,5,5,0:0,25,0,5:45:16:0:33:22:22:0:.\t0/1:0/1:16:9,3,3,1:0,12,0,4:9:16:9:32,34:23:23,23:2:16\r\n",
      "chr1\t1182895\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:19:12,7,0,0:0,19,0,0:84:16:0:34:60:60:0:.\t0/1:0/1:11:8,1,1,1:0,9,0,2:9:16:9:36,34:60:60,60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e7016f32-0703-4dd4-874d-cf41492e0275/e7016f32-0703-4dd4-874d-cf41492e0275.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160310\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job27850380_index__72vqN/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t877037\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:2,0,0,0:33:53:0:33:60:60:0:.\t0/1:0/1:7:1,1,4,1:2,0,5,0:15:53:46:31,33:34:60,24:2:16\r\n",
      "chr1\t877045\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,0,2,0:33:44:0:33:60:60:0:.\t0/1:0/1:6:1,1,3,1:4,0,2,0:21:44:37:31,33:39:29,60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a0cf7db2-fb3d-4811-b3d0-053c2d4d4fa2/a0cf7db2-fb3d-4811-b3d0-053c2d4d4fa2.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3d750d33-3567-4bea-bad5-a67c22ee6be6/3d750d33-3567-4bea-bad5-a67c22ee6be6.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job922796111_index_RDbJ4E/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t17556\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,2,0,0:33:35:0:33:34:34:0:.\t0/1:1/1:3:0,1,1,1:0,1,0,2:26:35:33:32:27:36:2:16\r\n",
      "chr1\t133194\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:2,2,0,0:0,4,0,0:39:18:0:35:51:51:0:.\t0/1:0/1:10:8,0,1,1:2,8,0,0:11:18:11:34,33:43:60,38:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/409a84e8-f202-4354-9561-262ef5ba3352/409a84e8-f202-4354-9561-262ef5ba3352.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/02d0d05a-7527-44d9-9715-fedf78ea7e19/02d0d05a-7527-44d9-9715-fedf78ea7e19.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job851047621_index_zRctpO/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t944350\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:0,2,0,0:0,2,0,0:33:35:0:27:60:60:0:.\t0/1:0/1:4:0,2,1,1:2,2,0,0:11:35:28:33,27:60:60,60:2:16\r\n",
      "chr1\t1068437\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:2,3,0,0:0,0,5,0:42:21:0:30:60:60:0:.\t0/1:1/1:3:1,0,1,1:2,0,1,0:15:21:29:25:60:60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/543408ab-1bb4-47f1-a59a-3dd5b0ea14d9/543408ab-1bb4-47f1-a59a-3dd5b0ea14d9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/742c1e15-01b9-471b-b01d-6198d48079a8/742c1e15-01b9-471b-b01d-6198d48079a8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0ce0240d-6a3e-46d8-a795-d2872fcfd46d/0ce0240d-6a3e-46d8-a795-d2872fcfd46d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2474fbd3-5e12-49f0-b596-6dc294ace8af/2474fbd3-5e12-49f0-b596-6dc294ace8af.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job879639388_index_ulBsRR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t835465\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:1,3,0,0:0,0,4,0:39:28:0:32:43:43:0:.\t0/1:1/1:2:0,0,1,1:2,0,0,0:33:28:33:31:27:27:2:21\r\n",
      "chr1\t971144\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:9,5,0,0:0,0,14,0:69:193:0:26:60:60:0:.\t0/1:0/1:26:7,6,8,5:13,0,13,0:185:193:186:32,31:60:60,60:2:52\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a7707f60-f9b9-4242-8b43-7ee28f31efa8/a7707f60-f9b9-4242-8b43-7ee28f31efa8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job356272531_index_jEU44c/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t13273\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:42:0:31:24:24:0:.\t0/1:1/1:5:1,1,2,1:0,3,2,0:3:42:36:32:27:32:2:16\r\n",
      "chr1\t14653\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:13:7,6,0,0:0,13,0,0:66:18:0:26:21:21:0:.\t0/1:0/1:13:8,1,4,0:0,9,0,4:11:18:11:32,34:22:22,23:2:18\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5e2c2661-7d51-48f8-a824-978fe1b9ecfd/5e2c2661-7d51-48f8-a824-978fe1b9ecfd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d2d7622c-97cd-4d0b-bdbe-9f9302d8ba70/d2d7622c-97cd-4d0b-bdbe-9f9302d8ba70.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job181350658_index_u0SrZu/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t186536\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,4,0,0:39:31:0:27:18:18:0:.\t0/1:1/1:6:0,2,0,4:0,2,0,4:13:31:37:33:19:21:2:21\r\n",
      "chr1\t1087713\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:33:31:0:30:60:60:0:.\t0/1:1/1:2:0,0,1,1:0,2,0,0:33:31:33:28:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b979479e-edaa-4ef8-955d-0378f9b52c0b/b979479e-edaa-4ef8-955d-0378f9b52c0b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3cdb5155-534c-4a57-bde1-9dcd225ee0e3/3cdb5155-534c-4a57-bde1-9dcd225ee0e3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/03541f9c-ece8-46a5-b060-4fbcbeae52ad/03541f9c-ece8-46a5-b060-4fbcbeae52ad.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job519632517_index_Aacmsh/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t942612\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:2,3,0,0:0,5,0,0:42:33:0:33:60:60:0:.\t0/1:1/1:3:0,1,1,1:0,1,0,2:5:33:31:31:60:60:2:24\r\n",
      "chr1\t1214147\t.\tA\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:24:11,13,0,0:24,0,0,0:99:24:0:33:60:60:0:.\t0/1:0/1:8:1,5,1,1:6,0,0,2:17:24:17:32,34:60:60,60:2:24\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d0e799b5-3c1f-4d6c-97d1-05a3fa421321/d0e799b5-3c1f-4d6c-97d1-05a3fa421321.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/257ef2a3-d03b-44a4-ba73-8fdffae58dd2/257ef2a3-d03b-44a4-ba73-8fdffae58dd2.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job672146980_index_zRFpMT/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14653\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:21:13,6,2,0:0,19,0,2:53:39:0:29:19:19:0:.\t0/1:0/1:21:10,4,6,1:0,14,0,7:32:39:32:29,31:17:14,22:2:34\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:195:72,103,10,10:20,0,175,0:216:69:0:30:51:54:0:.\t0/1:0/1:152:57,60,17,18:33,0,117,0:62:69:62:33,29:47:26,53:2:69\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/daf0b9d4-344b-4a7f-a806-708a60163d8a/daf0b9d4-344b-4a7f-a806-708a60163d8a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8186800e-cd84-40a8-aacd-4017db8b5c01/8186800e-cd84-40a8-aacd-4017db8b5c01.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/303d8054-5ea4-4330-9e2d-0870649b6b94/303d8054-5ea4-4330-9e2d-0870649b6b94.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6b3e88d4-72fe-4cf1-93af-0cb43849bcbc/6b3e88d4-72fe-4cf1-93af-0cb43849bcbc.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job877222183_index_vCjke9/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14653\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:20:10,9,1,0:0,19,0,1:64:18:0:27:18:17:0:.\t0/1:0/1:11:5,1,4,1:0,6,0,5:4:18:11:23,31:15:15,16:2:18\r\n",
      "chr1\t1042118\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:20:10,10,0,0:0,0,20,0:87:23:0:30:60:60:0:.\t0/1:0/1:6:1,3,1,1:0,0,4,2:16:23:16:33,30:60:60,60:2:23\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a99e4587-3414-4c10-a4ba-604bb0300bb0/a99e4587-3414-4c10-a4ba-604bb0300bb0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/68e55d82-f352-4f1d-8581-fe2d5a8dbd8f/68e55d82-f352-4f1d-8581-fe2d5a8dbd8f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3298901a-d3d1-47f2-a4fc-d516c5458caf/3298901a-d3d1-47f2-a4fc-d516c5458caf.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job771748187_index_Eol4pK/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1082764\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:4,0,0,0:0,0,0,4:39:33:0:26:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,2,0,0:33:33:33:32:60:60:2:22\r\n",
      "chr1\t1314344\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:16:9,7,0,0:0,0,16,0:75:19:0:33:60:60:0:.\t0/1:0/1:10:6,2,1,1:0,0,8,2:12:19:12:34,34:60:60,60:2:19\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/84590cda-2777-471b-badb-60bf023c75de/84590cda-2777-471b-badb-60bf023c75de.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4a9c2c7e-20bd-46e5-8362-3b40c01002ca/4a9c2c7e-20bd-46e5-8362-3b40c01002ca.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job166420704_index_YxtUa8/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t964905\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:52:0:35:60:60:0:.\t0/1:1/1:3:0,0,3,0:0,0,0,3:36:52:36:32:60:60:2:16\r\n",
      "chr1\t964906\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:2,0,0,0:33:63:0:33:60:60:0:.\t0/1:1/1:4:1,0,3,0:1,0,3,0:28:63:36:37:46:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c9ff5453-4017-40be-88ab-f1e2a5058537/c9ff5453-4017-40be-88ab-f1e2a5058537.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job472770305_index_GQdVBI/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1180954\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:3,2,0,0:0,0,5,0:42:40:0:24:56:56:0:.\t0/1:1/1:2:0,0,1,1:0,2,0,0:33:40:33:33:55:55:2:25\r\n",
      "chr1\t1180957\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:3,2,0,0:0,5,0,0:42:33:0:24:56:56:0:.\t0/1:1/1:4:1,1,1,1:0,2,0,2:5:33:31:32:51:55:2:24\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2cc47f4a-bffd-4ef9-b025-a03cb79eb4e1/2cc47f4a-bffd-4ef9-b025-a03cb79eb4e1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a648b9c4-55bf-4e3d-8de0-38b7b3354a31/a648b9c4-55bf-4e3d-8de0-38b7b3354a31.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2681a0ff-b4f9-4bfb-b0b0-e4a23a990850/2681a0ff-b4f9-4bfb-b0b0-e4a23a990850.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job61175832_index_x9p_i6/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1246652\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:40:0:29:60:60:0:.\t0/1:1/1:2:0,0,1,1:0,0,0,2:33:40:33:33:60:60:2:16\r\n",
      "chr1\t1455542\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:33:9,21,1,2:3,0,30,0:57:22:0:29:46:47:0:.\t0/1:0/1:32:6,20,3,3:6,0,26,0:15:22:15:30,31:43:26,47:2:22\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/ae40d22d-87c6-449f-a065-68a5cb26fc44/ae40d22d-87c6-449f-a065-68a5cb26fc44.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/96a9bc29-ad05-4493-bcf4-6d70975dddc3/96a9bc29-ad05-4493-bcf4-6d70975dddc3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bfcc2678-16d7-4e67-9130-057b0476bdb1/bfcc2678-16d7-4e67-9130-057b0476bdb1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/eb09f0f1-a1d8-411d-bb77-eecd038599ef/eb09f0f1-a1d8-411d-bb77-eecd038599ef.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/94e645ae-366e-4aca-a5db-e80800a33707/94e645ae-366e-4aca-a5db-e80800a33707.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/693cc9a1-20fb-469c-b57b-c5d2862e8886/693cc9a1-20fb-469c-b57b-c5d2862e8886.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/1588b2ec-2863-4ccb-aeda-b73b9195de76/1588b2ec-2863-4ccb-aeda-b73b9195de76.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/86ae836a-eed6-4fa1-9dd2-00ed4b5e9ca1/86ae836a-eed6-4fa1-9dd2-00ed4b5e9ca1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job516018207_index_lSy2cx/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1024082\t.\tT\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:1,4,0,0:0,0,0,5:42:27:0:32:60:60:0:.\t0/1:1/1:3:1,0,1,1:2,0,0,1:4:27:24:28:60:60:2:23\r\n",
      "chr1\t1427406\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:1,2,0,0:0,0,0,3:36:60:0:28:60:60:0:.\t0/1:1/1:4:0,1,2,1:0,3,0,1:28:60:36:33:53:50:2:19\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3f6b8dc3-74af-4677-9c46-59f424f8b8b4/3f6b8dc3-74af-4677-9c46-59f424f8b8b4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job171552488_index_7EW6fQ/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t966426\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:5,0,0,0:0,0,5,0:42:34:0:34:60:60:0:.\t0/1:0/1:4:2,0,2,0:0,2,2,0:25:34:27:35,34:60:60,60:2:24\r\n",
      "chr1\t999842\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:8:1,7,0,0:0,8,0,0:51:27:0:34:60:60:0:.\t0/1:0/1:11:0,8,0,3:3,8,0,0:20:27:20:30,29:60:60,60:2:26\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/59eeffbd-8211-420c-a3bd-baced95416a6/59eeffbd-8211-420c-a3bd-baced95416a6.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/291a955e-cfe4-4d12-bb40-c21fc2834371/291a955e-cfe4-4d12-bb40-c21fc2834371.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f6e5c3e0-99bf-4c2f-954e-850c32d9e910/f6e5c3e0-99bf-4c2f-954e-850c32d9e910.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160310\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job500263790_index_TyhtB_/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t15820\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:34:12,17,5,0:0,0,29,5:49:83:0:33:32:35:0:.\t0/1:0/1:32:11,10,11,0:0,0,21,11:76:83:76:33,33:35:36,34:2:32\r\n",
      "chr1\t187744\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:0,3,0,0:3,0,0,0:36:17:0:23:23:23:0:.\t0/1:1/1:2:0,0,0,2:0,0,2,0:33:17:33:33:23:23:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b92607dc-525a-4695-a47c-071cd37de5f7/b92607dc-525a-4695-a47c-071cd37de5f7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/8dc2f88f-90e0-4790-b1dc-7b10eeca4104/8dc2f88f-90e0-4790-b1dc-7b10eeca4104.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3b559cb3-81f9-4155-af8a-2dd7013d5ed8/3b559cb3-81f9-4155-af8a-2dd7013d5ed8.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job78751889_index_8uMMaK/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t187497\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:95:55,25,12,3:15,0,80,0:60:83:0:28:27:27:0:.\t0/1:0/1:87:44,22,12,9:21,0,66,0:76:83:76:31,29:28:28,29:2:43\r\n",
      "chr1\t971027\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:13:8,5,0,0:0,13,0,0:66:25:0:31:60:60:0:.\t0/1:0/1:7:2,3,1,1:0,5,0,2:18:25:18:18,33:60:60,60:2:25\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/90a63527-ec26-454c-81b7-ac68007ad0eb/90a63527-ec26-454c-81b7-ac68007ad0eb.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job999149087_index_0cGU9j/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t939519\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:0,13,0,1:0,13,1,0:38:19:0:33:60:60:0:.\t0/1:0/1:9:0,7,1,1:2,7,0,0:12:19:12:33,32:61:60,61:2:19\r\n",
      "chr1\t1263119\t.\tT\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:17:13,0,4,0:0,1,3,13:56:16:0:23:60:60:0:.\t0/1:1/1:8:1,0,7,0:0,0,7,1:21:16:30:10:55:55:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d5f9c363-f568-406c-8ee6-2ed3f4bf796d/d5f9c363-f568-406c-8ee6-2ed3f4bf796d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160306\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job584977028_index_O73ikl/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t874301\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:133:58,58,9,8:17,0,116,0:115:62:0:28:51:55:0:.\t0/1:0/1:70:24,27,10,9:19,0,51,0:55:62:55:34,30:44:18,54:2:62\r\n",
      "chr1\t874307\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:132:54,61,9,8:0,17,115,0:124:64:0:32:52:57:0:.\t0/1:0/1:67:27,21,10,9:0,19,48,0:57:64:57:31,33:46:18,57:2:64\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5a10ae7c-0e93-4159-9628-be3da9b766c3/5a10ae7c-0e93-4159-9628-be3da9b766c3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f896d719-8fb7-41fa-98e3-dbf7a7f4532a/f896d719-8fb7-41fa-98e3-dbf7a7f4532a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2905fd44-babc-455c-a8d3-5da61b96e6b0/2905fd44-babc-455c-a8d3-5da61b96e6b0.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160304\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job443410469_index_hVa5YW/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t931147\t.\tA\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:16:0,14,1,1:14,1,0,1:43:20:0:29:60:60:0:.\t0/1:0/1:14:0,10,1,3:10,1,0,3:13:20:13:26,31:60:60,60:2:19\r\n",
      "chr1\t942637\t.\tC\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,2,0,0:33:31:0:32:60:60:0:.\t0/1:1/1:2:0,0,1,1:2,0,0,0:33:31:33:28:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/41783482-0067-4f23-b7f1-24b38d2e7bf1/41783482-0067-4f23-b7f1-24b38d2e7bf1.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/792602af-825a-4b2a-a684-eb9573ba962a/792602af-825a-4b2a-a684-eb9573ba962a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/2d48c751-1a4c-41a3-9cdb-6afdb3f3e5c7/2d48c751-1a4c-41a3-9cdb-6afdb3f3e5c7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e6ccb762-b6f8-4e34-86f6-8bed7cf03a5a/e6ccb762-b6f8-4e34-86f6-8bed7cf03a5a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job330226514_index_EzhFUg/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:10,4,0,0:0,14,0,0:69:21:0:33:20:20:0:.\t0/1:0/1:16:8,5,1,2:0,13,3,0:14:21:14:33,34:24:22,37:2:21\r\n",
      "chr1\t1637362\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:6:5,0,1,0:0,5,1,0:30:31:0:29:55:54:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:24:31:24:33,33:56:53,60:2:24\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/90d23218-8c42-4d47-a779-545b624810c3/90d23218-8c42-4d47-a779-545b624810c3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/0f238983-ce97-42eb-9fb1-bc2dc27cdda3/0f238983-ce97-42eb-9fb1-bc2dc27cdda3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job12756511_index_6L6Ruh/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t185428\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:3:1,2,0,0:3,0,0,0:36:18:0:31:29:29:0:.\t0/1:1/1:2:0,0,0,2:0,0,2,0:33:18:33:31:23:23:2:15\r\n",
      "chr1\t908746\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:2,2,0,0:4,0,0,0:39:21:0:24:47:47:0:.\t0/1:1/1:3:1,0,1,1:1,0,2,0:5:21:19:33:42:33:2:18\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/cd533315-a49b-4d5d-ac36-748d09aa2a88/cd533315-a49b-4d5d-ac36-748d09aa2a88.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job688015949_index_UmYlvb/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t765221\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,4,0,0:39:37:0:34:15:15:0:.\t0/1:0/1:8:0,4,0,4:0,4,0,4:30:37:30:34,29:27:28,25:2:22\r\n",
      "chr1\t1050937\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:8:4,4,0,0:0,0,0,8:51:35:0:32:55:55:0:.\t0/1:1/1:2:0,0,1,1:0,2,0,0:33:35:33:30:60:60:2:31\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/194e6508-7d99-4cfb-8bf5-f036473aa76c/194e6508-7d99-4cfb-8bf5-f036473aa76c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d0cc82cf-52c6-4b72-864a-2691cb7878a2/d0cc82cf-52c6-4b72-864a-2691cb7878a2.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/5c598334-1e51-4793-b029-cc5903e185ae/5c598334-1e51-4793-b029-cc5903e185ae.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a14f5ad5-ff5d-4d96-abc7-d4a55cd683b7/a14f5ad5-ff5d-4d96-abc7-d4a55cd683b7.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/3b66f4aa-ffe6-448d-8948-ff0322fe6bda/3b66f4aa-ffe6-448d-8948-ff0322fe6bda.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/f534ac38-208b-4386-87c6-df99e3dcbd73/f534ac38-208b-4386-87c6-df99e3dcbd73.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/74c2a655-15e8-4cb7-b6fe-d07decf92c70/74c2a655-15e8-4cb7-b6fe-d07decf92c70.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/71aed459-18b1-4d07-b00b-7852f5937f0a/71aed459-18b1-4d07-b00b-7852f5937f0a.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160305\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job729510122_index_v9ifsR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1050070\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:8:4,4,0,0:0,0,8,0:51:29:0:16:32:32:0:.\t0/1:0/1:18:9,5,4,0:4,0,14,0:22:29:22:31,11:33:60,26:2:28\r\n",
      "chr1\t1071904\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:1,1,0,0:0,0,2,0:33:40:0:30:60:60:0:.\t0/1:1/1:2:0,0,1,1:0,0,0,2:33:40:33:33:60:60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/18fdd607-4b46-4dd5-863e-ff40a80e4b0c/18fdd607-4b46-4dd5-863e-ff40a80e4b0c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job461517664_index_v4A5eX/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t187497\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:61:38,18,5,0:5,0,56,0:114:53:0:28:26:27:0:.\t0/1:0/1:66:38,14,9,5:14,0,52,0:46:53:46:33,29:26:26,25:2:53\r\n",
      "chr1\t1183055\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:11:1,10,0,0:0,0,11,0:60:15:0:31:60:60:0:.\t0/1:0/1:17:3,11,1,2:0,0,14,3:8:15:8:32,30:60:60,60:2:15\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d689b050-7d3a-48bb-8e6a-125239c364db/d689b050-7d3a-48bb-8e6a-125239c364db.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job630016258_index_dYVOxS/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t602209\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:33:25:0:34:19:19:0:.\t0/1:1/1:2:0,0,0,2:0,2,0,0:33:25:33:30:28:28:2:15\r\n",
      "chr1\t863797\t.\tG\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,2,0:33:33:0:31:60:60:0:.\t0/1:0/1:4:2,0,2,0:0,2,2,0:23:33:26:35,33:54:48,60:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/83143db6-9bb4-420a-a7c5-2abce396da2e/83143db6-9bb4-420a-a7c5-2abce396da2e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b2cb90b0-6ff5-42b7-96c2-f10df30f470d/b2cb90b0-6ff5-42b7-96c2-f10df30f470d.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160316\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job705649442_index_AA_Iov/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t964383\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:2,2,0,0:0,0,4,0:39:33:0:32:60:60:0:.\t0/1:0/1:4:1,1,1,1:0,0,2,2:21:33:26:30,32:60:60,60:2:22\r\n",
      "chr1\t1048976\t.\tT\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:5,9,0,0:0,0,0,14:69:21:0:28:60:60:0:.\t0/1:0/1:8:4,2,1,1:2,0,0,6:14:21:14:32,31:60:60,60:2:21\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/535f03d3-a012-4ff5-a75d-96e31445a6b4/535f03d3-a012-4ff5-a75d-96e31445a6b4.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!cat head.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [],
   "source": [
    "tmpStr=str(file_content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ordinary string\n"
     ]
    }
   ],
   "source": [
    "def whatisthis(s):\n",
    "    if isinstance(s, str):\n",
    "        print (\"ordinary string\")\n",
    "    elif isinstance(s, unicode):\n",
    "        print (\"unicode string\")\n",
    "    else:\n",
    "        print (\"not a string\")\n",
    "whatisthis(file_content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 113,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "'fileformat' in file_content"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'##fileform'"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "file_content[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##fileformat=VCFv4.1\r\n",
      "##FILTER=<ID=PASS,Description=\"Accept as a confident somatic mutation\">\r\n",
      "##FILTER=<ID=Tier1,Description=\"Confident level 1\">\r\n",
      "##FILTER=<ID=Tier2,Description=\"Confident level 2\">\r\n",
      "##FILTER=<ID=Tier3,Description=\"Confident level 3\">\r\n",
      "##FILTER=<ID=Tier4,Description=\"Confident level 4\">\r\n",
      "##FILTER=<ID=Tier5,Description=\"Confident level 5\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read depth at this position in the sample\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Depth of reads supporting alleles 0/1/2/3...\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality for reads supporting alleles\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal,0=wildtype,1=germline,2=somatic,3=LOH,4=post-transcriptional modification,5=unknown\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##TUMOR=\"Sample=TCGA-DU-5874-01A-11D-1705-08,File=/var/lib/cwl/job178108365_input_5fUL1E/C494.TCGA-DU-5874-01A-11D-1705-08.5_gdc_realn.bam\"\r\n",
      "##NORMAL=\"Sample=TCGA-DU-5874-10A-01D-1705-08,File=/var/lib/cwl/job178108365_input_5fUL1E/C494.TCGA-DU-5874-10A-01D-1705-08.5_gdc_realn.bam\"\r\n",
      "##MuSE_version=\"v1.0rc Build Date Aug  6 2015 Build Time 10:11:56\"\r\n",
      "##MuSE_call_1=\"call -f /var/lib/cwl/job178108365_index_0nC40p/GRCh38.d1.vd1.fa -r chr7:1-50000000 -O /var/spool/cwl/call/output.0 /var/lib/cwl/job178108365_input_5fUL1E/C494.TCGA-DU-5874-01A-11D-1705-08.5_gdc_realn.bam /var/lib/cwl/job178108365_input_5fUL1E/C494.TCGA-DU-5874-10A-01D-1705-08.5_gdc_realn.bam\"\r\n",
      "##MuSE_sump=\"sump -I /var/lib/cwl/job408821879_workdir_99LxilPea478/C494.TCGA-DU-5874-01A-11D-1705-08.5_gdc_realn.MuSE.txt -E -O 09d7a024-9dee-4d4c-8591-be09cdbdfb77.vcf -D /var/lib/cwl/job408821879_index_0nC40p/dbsnp_144.grch38.vcf.bgz\"\r\n",
      "##contig=<ID=chr1,length=248956422,assembly=GRCh38.d1.vd1>\r\n",
      "##contig=<ID=chr2,length=242193529,assembly=GRCh38.d1.vd1>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##FILTER=<ID=PASS,Description=\"Accept as a confident somatic mutation\">\r\n",
      "##FILTER=<ID=Tier1,Description=\"Confident level 1\">\r\n",
      "##FILTER=<ID=Tier2,Description=\"Confident level 2\">\r\n",
      "##FILTER=<ID=Tier3,Description=\"Confident level 3\">\r\n",
      "##FILTER=<ID=Tier4,Description=\"Confident level 4\">\r\n",
      "##FILTER=<ID=Tier5,Description=\"Confident level 5\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read depth at this position in the sample\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Depth of reads supporting alleles 0/1/2/3...\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality for reads supporting alleles\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal,0=wildtype,1=germline,2=somatic,3=LOH,4=post-transcriptional modification,5=unknown\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##TUMOR=\"Sample=TCGA-TQ-A7RG-01A-11D-A33T-08,File=/var/lib/cwl/job214646024_input_3HzcpP/C494.TCGA-TQ-A7RG-01A-11D-A33T-08.1_gdc_realn.bam\"\r\n",
      "##NORMAL=\"Sample=TCGA-TQ-A7RG-10A-01D-A33W-08,File=/var/lib/cwl/job214646024_input_3HzcpP/C494.TCGA-TQ-A7RG-10A-01D-A33W-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_version=\"v1.0rc Build Date Aug  6 2015 Build Time 10:11:56\"\r\n",
      "##MuSE_call_1=\"call -f /var/lib/cwl/job214646024_index_TqPhA6/GRCh38.d1.vd1.fa -r chr18:1-50000000 -O /var/spool/cwl/call/output.0 /var/lib/cwl/job214646024_input_3HzcpP/C494.TCGA-TQ-A7RG-01A-11D-A33T-08.1_gdc_realn.bam /var/lib/cwl/job214646024_input_3HzcpP/C494.TCGA-TQ-A7RG-10A-01D-A33W-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_sump=\"sump -I /var/lib/cwl/job273135982_workdir_ndECT1HncOCk/C494.TCGA-TQ-A7RG-01A-11D-A33T-08.1_gdc_realn.MuSE.txt -E -O 6ecc68de-4217-419c-ad61-591e314e655e.vcf -D /var/lib/cwl/job273135982_index_TqPhA6/dbsnp_144.grch38.vcf.bgz\"\r\n",
      "##contig=<ID=chr1,length=248956422,assembly=GRCh38.d1.vd1>\r\n",
      "##contig=<ID=chr2,length=242193529,assembly=GRCh38.d1.vd1>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##FILTER=<ID=PASS,Description=\"Accept as a confident somatic mutation\">\r\n",
      "##FILTER=<ID=Tier1,Description=\"Confident level 1\">\r\n",
      "##FILTER=<ID=Tier2,Description=\"Confident level 2\">\r\n",
      "##FILTER=<ID=Tier3,Description=\"Confident level 3\">\r\n",
      "##FILTER=<ID=Tier4,Description=\"Confident level 4\">\r\n",
      "##FILTER=<ID=Tier5,Description=\"Confident level 5\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read depth at this position in the sample\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Depth of reads supporting alleles 0/1/2/3...\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality for reads supporting alleles\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal,0=wildtype,1=germline,2=somatic,3=LOH,4=post-transcriptional modification,5=unknown\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##TUMOR=\"Sample=TCGA-RY-A845-01A-11D-A36O-08,File=/var/lib/cwl/job472013018_input_fKCH9P/C494.TCGA-RY-A845-01A-11D-A36O-08.1_gdc_realn.bam\"\r\n",
      "##NORMAL=\"Sample=TCGA-RY-A845-10A-01D-A367-08,File=/var/lib/cwl/job472013018_input_fKCH9P/C494.TCGA-RY-A845-10A-01D-A367-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_version=\"v1.0rc Build Date Aug  6 2015 Build Time 10:11:56\"\r\n",
      "##MuSE_call_1=\"call -f /var/lib/cwl/job472013018_reference/GRCh38.d1.vd1.fa -r chr6:1-50000000 -O /var/spool/cwl/call/output.0 /var/lib/cwl/job472013018_input_fKCH9P/C494.TCGA-RY-A845-01A-11D-A36O-08.1_gdc_realn.bam /var/lib/cwl/job472013018_input_fKCH9P/C494.TCGA-RY-A845-10A-01D-A367-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_sump=\"sump -I /var/lib/cwl/job26216091_workdir_ixtr8Sjding6/C494.TCGA-RY-A845-01A-11D-A36O-08.1_gdc_realn.MuSE.txt -E -O 6a640b52-648e-4c47-b8a2-c80c7f813e4c.vcf -D /var/lib/cwl/job26216091_reference/dbsnp_144.grch38.vcf.bgz\"\r\n",
      "##contig=<ID=chr1,length=248956422,assembly=GRCh38.d1.vd1>\r\n",
      "##contig=<ID=chr2,length=242193529,assembly=GRCh38.d1.vd1>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##FILTER=<ID=PASS,Description=\"Accept as a confident somatic mutation\">\r\n",
      "##FILTER=<ID=Tier1,Description=\"Confident level 1\">\r\n",
      "##FILTER=<ID=Tier2,Description=\"Confident level 2\">\r\n",
      "##FILTER=<ID=Tier3,Description=\"Confident level 3\">\r\n",
      "##FILTER=<ID=Tier4,Description=\"Confident level 4\">\r\n",
      "##FILTER=<ID=Tier5,Description=\"Confident level 5\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read depth at this position in the sample\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Depth of reads supporting alleles 0/1/2/3...\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality for reads supporting alleles\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal,0=wildtype,1=germline,2=somatic,3=LOH,4=post-transcriptional modification,5=unknown\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##TUMOR=\"Sample=TCGA-HW-8321-01A-11D-2395-08,File=/var/lib/cwl/job413866067_input_vYlv1y/C494.TCGA-HW-8321-01A-11D-2395-08.1_gdc_realn.bam\"\r\n",
      "##NORMAL=\"Sample=TCGA-HW-8321-10A-01D-2396-08,File=/var/lib/cwl/job413866067_input_vYlv1y/C494.TCGA-HW-8321-10A-01D-2396-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_version=\"v1.0rc Build Date Aug  6 2015 Build Time 10:11:56\"\r\n",
      "##MuSE_call_1=\"call -f /var/lib/cwl/job413866067_reference/GRCh38.d1.vd1.fa -r chrM:1-16569 -O /var/spool/cwl/call/output.0 /var/lib/cwl/job413866067_input_vYlv1y/C494.TCGA-HW-8321-01A-11D-2395-08.1_gdc_realn.bam /var/lib/cwl/job413866067_input_vYlv1y/C494.TCGA-HW-8321-10A-01D-2396-08.1_gdc_realn.bam\"\r\n",
      "##MuSE_sump=\"sump -I /var/lib/cwl/job909254120_workdir_JNKeb_z15nC8/C494.TCGA-HW-8321-01A-11D-2395-08.1_gdc_realn.MuSE.txt -E -O /var/spool/cwl/57e048ed-5681-413e-bbdc-0ff202566744.vcf -D /var/lib/cwl/job909254120_reference/dbsnp_144.grch38.vcf.bgz\"\r\n",
      "##contig=<ID=chr1,length=248956422,assembly=GRCh38.d1.vd1>\r\n",
      "##contig=<ID=chr2,length=242193529,assembly=GRCh38.d1.vd1>\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/99690673-b98b-496c-bf5e-e08d3b3dc545/99690673-b98b-496c-bf5e-e08d3b3dc545.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job127557346_index_O09uzk/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1020217\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,2,0:33:36:0:26:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,0,0,2:33:36:33:33:60:60:2:16\r\n",
      "chr1\t1022591\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:9:0,9,0,0:0,0,9,0:54:60:0:33:52:52:0:.\t0/1:0/1:15:0,10,0,5:5,0,10,0:53:60:53:32,33:54:60,52:2:37\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c48297db-2dcd-4216-9033-f9bdbe433775/c48297db-2dcd-4216-9033-f9bdbe433775.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/dbfb00be-ed30-402f-bb2e-19e7d0ec340f/dbfb00be-ed30-402f-bb2e-19e7d0ec340f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4aa94453-1f1c-4120-864e-6b018f10704e/4aa94453-1f1c-4120-864e-6b018f10704e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/52615c7e-1274-4513-bd11-c063aef4ae0f/52615c7e-1274-4513-bd11-c063aef4ae0f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/99690673-b98b-496c-bf5e-e08d3b3dc545/99690673-b98b-496c-bf5e-e08d3b3dc545.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160307\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job127557346_index_O09uzk/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t1020217\t.\tG\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,2,0:33:36:0:26:60:60:0:.\t0/1:1/1:2:0,0,2,0:0,0,0,2:33:36:33:33:60:60:2:16\r\n",
      "chr1\t1022591\t.\tG\tA\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:9:0,9,0,0:0,0,9,0:54:60:0:33:52:52:0:.\t0/1:0/1:15:0,10,0,5:5,0,10,0:53:60:53:32,33:54:60,52:2:37\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c48297db-2dcd-4216-9033-f9bdbe433775/c48297db-2dcd-4216-9033-f9bdbe433775.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/dbfb00be-ed30-402f-bb2e-19e7d0ec340f/dbfb00be-ed30-402f-bb2e-19e7d0ec340f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/4aa94453-1f1c-4120-864e-6b018f10704e/4aa94453-1f1c-4120-864e-6b018f10704e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/52615c7e-1274-4513-bd11-c063aef4ae0f/52615c7e-1274-4513-bd11-c063aef4ae0f.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/eef4b992-9147-45d9-9530-eaf2fedb2015/eef4b992-9147-45d9-9530-eaf2fedb2015.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/b8d00849-003c-4675-b1ca-1382316a0c93/b8d00849-003c-4675-b1ca-1382316a0c93.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job752047669_index_yaoK3P/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t183800\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:14:6,7,1,0:0,13,1,0:47:33:0:33:22:22:0:.\t0/1:0/1:21:12,4,3,2:0,16,5,0:26:33:26:29,34:27:27,25:2:28\r\n",
      "chr1\t1290155\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,0,0,2:28:26:0:26:23:23:0:.\t0/1:1/1:5:3,0,2,0:0,2,0,3:10:26:29:33:31:47:2:16\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d061c5fb-87d1-451e-a257-eaee4e7af7a9/d061c5fb-87d1-451e-a257-eaee4e7af7a9.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/c6c7dae7-60be-4acc-b6c8-2866858d0194/c6c7dae7-60be-4acc-b6c8-2866858d0194.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/6f824911-c2d5-4c4b-b1a1-377fe4b8844e/6f824911-c2d5-4c4b-b1a1-377fe4b8844e.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/d6084f6a-122c-4efb-a2bf-ef50024373f3/d6084f6a-122c-4efb-a2bf-ef50024373f3.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160309\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job170879263_index_UUzyAz/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t933015\t.\tC\tT\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:2:2,0,0,0:0,2,0,0:33:23:0:32:60:60:0:.\t0/1:0/1:5:3,0,1,1:0,3,0,2:16:23:16:32,29:60:60,60:2:15\r\n",
      "chr1\t948731\t.\tT\tC\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:4:0,4,0,0:0,0,0,4:39:24:0:31:60:60:0:.\t0/1:0/1:10:0,7,0,3:0,3,0,7:17:24:17:28,29:60:60,60:2:20\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/51946e4b-3a0b-40d5-84b0-9c08f44c7098/51946e4b-3a0b-40d5-84b0-9c08f44c7098.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3b48fa3-a040-450f-9d33-efdac2537717/a3b48fa3-a040-450f-9d33-efdac2537717.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.1\r\n",
      "##fileDate=20160317\r\n",
      "##phasing=none\r\n",
      "##reference=file:///var/lib/cwl/job569970468_index_b3SOZR/GRCh38.d1.vd1.fa\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=IGT,Number=1,Type=String,Description=\"Genotype when called independently (only filled if called in joint prior mode)\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Total read depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=4,Type=Integer,Description=\"# high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases\">\r\n",
      "##FORMAT=<ID=BCOUNT,Number=4,Type=Integer,Description=\"Occurrence count for each base at this site (A,C,G,T)\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype quality\">\r\n",
      "##FORMAT=<ID=JGQ,Number=1,Type=Integer,Description=\"Joint genotype quality (only filled if called in join prior mode)\">\r\n",
      "##FORMAT=<ID=VAQ,Number=1,Type=Integer,Description=\"Variant allele quality\">\r\n",
      "##FORMAT=<ID=BQ,Number=.,Type=Integer,Description=\"Average base quality\">\r\n",
      "##FORMAT=<ID=MQ,Number=1,Type=Integer,Description=\"Average mapping quality across all reads\">\r\n",
      "##FORMAT=<ID=AMQ,Number=.,Type=Integer,Description=\"Average mapping quality for each allele present in the genotype\">\r\n",
      "##FORMAT=<ID=SS,Number=1,Type=Integer,Description=\"Variant status relative to non-adjacent Normal, 0=wildtype,1=germline,2=somatic,3=LOH,4=unknown\">\r\n",
      "##FORMAT=<ID=SSC,Number=1,Type=Integer,Description=\"Somatic Score\">\r\n",
      "#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\tNORMAL\tTUMOR\r\n",
      "chr1\t14574\t.\tA\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:15:13,0,2,0:13,0,2,0:47:19:0:33:21:23:0:.\t0/1:0/1:15:9,1,5,0:10,0,5,0:12:19:12:33,27:24:24,22:2:19\r\n",
      "chr1\t1312209\t.\tC\tG\t.\t.\t.\tGT:IGT:DP:DP4:BCOUNT:GQ:JGQ:VAQ:BQ:MQ:AMQ:SS:SSC\t0/0:0/0:5:0,5,0,0:0,5,0,0:42:18:0:10:39:39:0:.\t0/1:0/1:8:2,2,2,2:0,4,4,0:11:18:11:16,15:46:32,60:2:17\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd/e07cb0ce-48a1-42b7-9c21-63d2de64b8cd.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/44663ff0-2002-4e7d-a6ca-8c834f8698da/44663ff0-2002-4e7d-a6ca-8c834f8698da.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/bcd38101-c9e3-4d7e-8951-3a0434474187/bcd38101-c9e3-4d7e-8951-3a0434474187.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n",
      "/nrnb/users/btsui/Data/tcga_orig_vcf/a3650193-4824-4db6-bbb8-7471324b2d7b/a3650193-4824-4db6-bbb8-7471324b2d7b.vcf.gz\r\n",
      "\r\n",
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=alt_allele_in_normal,Description=\"Evidence seen in the normal sample\">\r\n",
      "##FILTER=<ID=clustered_events,Description=\"Clustered events observed in the tumor\">\r\n",
      "##FILTER=<ID=germline_risk,Description=\"Evidence indicates this site is germline, not somatic\">\r\n",
      "##FILTER=<ID=homologous_mapping_event,Description=\"More than three events were observed in the tumor\">\r\n",
      "##FILTER=<ID=multi_event_alt_allele_in_normal,Description=\"Multiple events observed in tumor and normal\">\r\n",
      "##FILTER=<ID=panel_of_normals,Description=\"Seen in at least 2 samples in the panel of normals\">\r\n",
      "##FILTER=<ID=str_contraction,Description=\"Site filtered due to contraction of short tandem repeat region\">\r\n",
      "##FILTER=<ID=t_lod_fstar,Description=\"Tumor does not meet likelihood threshold\">\r\n",
      "##FILTER=<ID=triallelic_site,Description=\"Site filtered because more than two alt alleles pass tumor LOD\">\r\n",
      "##FORMAT=<ID=AD,Number=.,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=AF,Number=1,Type=Float,Description=\"Allele fraction of the event in the tumor\">\r\n",
      "##FORMAT=<ID=ALT_F1R2,Number=1,Type=Integer,Description=\"Count of reads in F1R2 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=ALT_F2R1,Number=1,Type=Integer,Description=\"Count of reads in F2R1 pair orientation supporting the alternate allele\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">\r\n",
      "##FORMAT=<ID=FOXOG,Number=1,Type=Float,Description=\"Fraction of alt reads indicating OxoG error\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=PGT,Number=1,Type=String,Description=\"Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another\">\r\n",
      "##FORMAT=<ID=PID,Number=1,Type=String,Description=\"Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group\">\r\n",
      "\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!cat head.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "176 head.txt\r\n"
     ]
    }
   ],
   "source": [
    "!wc -l head.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##fileformat=VCFv4.2\r\n",
      "##FILTER=<ID=indelError,Description=\"Likely artifact due to indel reads at this position\">\r\n",
      "##FILTER=<ID=str10,Description=\"Less than 10% or more than 90% of variant supporting reads on one strand\">\r\n",
      "##FORMAT=<ID=AD,Number=R,Type=Integer,Description=\"Allelic depths for the ref and alt alleles in the order listed\">\r\n",
      "##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Read Depth\">\r\n",
      "##FORMAT=<ID=DP4,Number=1,Type=String,Description=\"Strand read counts: ref/fwd, ref/rev, var/fwd, var/rev\">\r\n",
      "##FORMAT=<ID=FREQ,Number=1,Type=String,Description=\"Variant allele frequency\">\r\n",
      "##FORMAT=<ID=GQ,Number=1,Type=Integer,Description=\"Genotype Quality\">\r\n",
      "##FORMAT=<ID=GT,Number=1,Type=String,Description=\"Genotype\">\r\n",
      "##FORMAT=<ID=RD,Number=1,Type=Integer,Description=\"Depth of reference-supporting bases (reads1)\">\r\n",
      "##INFO=<ID=DP,Number=1,Type=Integer,Description=\"Total depth of quality bases\">\r\n",
      "##INFO=<ID=GPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor+normal versus no variant for Germline calls\">\r\n",
      "##INFO=<ID=SOMATIC,Number=0,Type=Flag,Description=\"Indicates if record is a somatic mutation\">\r\n",
      "##INFO=<ID=SPV,Number=1,Type=Float,Description=\"Fisher's Exact Test P-value of tumor versus normal for Somatic/LOH calls\">\r\n",
      "##INFO=<ID=SS,Number=1,Type=String,Description=\"Somatic status of variant (0=Reference,1=Germline,2=Somatic,3=LOH, or 5=Unknown)\">\r\n",
      "##INFO=<ID=SSC,Number=1,Type=String,Description=\"Somatic score in Phred scale (0-255) derived from somatic p-value\">\r\n",
      "##contig=<ID=chr1,length=248956422>\r\n",
      "##contig=<ID=chr2,length=242193529>\r\n",
      "##contig=<ID=chr3,length=198295559>\r\n",
      "##contig=<ID=chr4,length=190214555>\r\n",
      "##contig=<ID=chr5,length=181538259>\r\n",
      "##contig=<ID=chr6,length=170805979>\r\n",
      "##contig=<ID=chr7,length=159345973>\r\n",
      "##contig=<ID=chr8,length=145138636>\r\n",
      "##contig=<ID=chr9,length=138394717>\r\n",
      "##contig=<ID=chr10,length=133797422>\r\n",
      "##contig=<ID=chr11,length=135086622>\r\n",
      "##contig=<ID=chr12,length=133275309>\r\n",
      "##contig=<ID=chr13,length=114364328>\r\n",
      "##contig=<ID=chr14,length=107043718>\r\n",
      "##contig=<ID=chr15,length=101991189>\r\n",
      "##contig=<ID=chr16,length=90338345>\r\n",
      "##contig=<ID=chr17,length=83257441>\r\n",
      "##contig=<ID=chr18,length=80373285>\r\n",
      "##contig=<ID=chr19,length=58617616>\r\n",
      "##contig=<ID=chr20,length=64444167>\r\n",
      "##contig=<ID=chr21,length=46709983>\r\n",
      "##contig=<ID=chr22,length=50818468>\r\n",
      "##contig=<ID=chrX,length=156040895>\r\n",
      "##contig=<ID=chrY,length=57227415>\r\n",
      "##contig=<ID=chrM,length=16569>\r\n",
      "##contig=<ID=chr1_KI270706v1_random,length=175055>\r\n",
      "##contig=<ID=chr1_KI270707v1_random,length=32032>\r\n",
      "##contig=<ID=chr1_KI270708v1_random,length=127682>\r\n",
      "##contig=<ID=chr1_KI270709v1_random,length=66860>\r\n",
      "##contig=<ID=chr1_KI270710v1_random,length=40176>\r\n",
      "##contig=<ID=chr1_KI270711v1_random,length=42210>\r\n",
      "##contig=<ID=chr1_KI270712v1_random,length=176043>\r\n",
      "##contig=<ID=chr1_KI270713v1_random,length=40745>\r\n",
      "##contig=<ID=chr1_KI270714v1_random,length=41717>\r\n",
      "##contig=<ID=chr2_KI270715v1_random,length=161471>\r\n",
      "##contig=<ID=chr2_KI270716v1_random,length=153799>\r\n",
      "##contig=<ID=chr3_GL000221v1_random,length=155397>\r\n",
      "##contig=<ID=chr4_GL000008v2_random,length=209709>\r\n",
      "##contig=<ID=chr5_GL000208v1_random,length=92689>\r\n",
      "##contig=<ID=chr9_KI270717v1_random,length=40062>\r\n",
      "##contig=<ID=chr9_KI270718v1_random,length=38054>\r\n",
      "##contig=<ID=chr9_KI270719v1_random,length=176845>\r\n",
      "##contig=<ID=chr9_KI270720v1_random,length=39050>\r\n",
      "##contig=<ID=chr11_KI270721v1_random,length=100316>\r\n",
      "##contig=<ID=chr14_GL000009v2_random,length=201709>\r\n",
      "##contig=<ID=chr14_GL000225v1_random,length=211173>\r\n",
      "##contig=<ID=chr14_KI270722v1_random,length=194050>\r\n",
      "##contig=<ID=chr14_GL000194v1_random,length=191469>\r\n",
      "##contig=<ID=chr14_KI270723v1_random,length=38115>\r\n",
      "##contig=<ID=chr14_KI270724v1_random,length=39555>\r\n",
      "##contig=<ID=chr14_KI270725v1_random,length=172810>\r\n",
      "##contig=<ID=chr14_KI270726v1_random,length=43739>\r\n",
      "##contig=<ID=chr15_KI270727v1_random,length=448248>\r\n",
      "##contig=<ID=chr16_KI270728v1_random,length=1872759>\r\n",
      "##contig=<ID=chr17_GL000205v2_random,length=185591>\r\n",
      "##contig=<ID=chr17_KI270729v1_random,length=280839>\r\n",
      "##contig=<ID=chr17_KI270730v1_random,length=112551>\r\n",
      "##contig=<ID=chr22_KI270731v1_random,length=150754>\r\n",
      "##contig=<ID=chr22_KI270732v1_random,length=41543>\r\n",
      "##contig=<ID=chr22_KI270733v1_random,length=179772>\r\n",
      "##contig=<ID=chr22_KI270734v1_random,length=165050>\r\n",
      "##contig=<ID=chr22_KI270735v1_random,length=42811>\r\n",
      "##contig=<ID=chr22_KI270736v1_random,length=181920>\r\n",
      "##contig=<ID=chr22_KI270737v1_random,length=103838>\r\n",
      "##contig=<ID=chr22_KI270738v1_random,length=99375>\r\n",
      "##contig=<ID=chr22_KI270739v1_random,length=73985>\r\n",
      "##contig=<ID=chrY_KI270740v1_random,length=37240>\r\n",
      "##contig=<ID=chrUn_KI270302v1,length=2274>\r\n",
      "##contig=<ID=chrUn_KI270304v1,length=2165>\r\n",
      "##contig=<ID=chrUn_KI270303v1,length=1942>\r\n",
      "##contig=<ID=chrUn_KI270305v1,length=1472>\r\n",
      "##contig=<ID=chrUn_KI270322v1,length=21476>\r\n",
      "##contig=<ID=chrUn_KI270320v1,length=4416>\r\n",
      "##contig=<ID=chrUn_KI270310v1,length=1201>\r\n",
      "##contig=<ID=chrUn_KI270316v1,length=1444>\r\n",
      "##contig=<ID=chrUn_KI270315v1,length=2276>\r\n",
      "##contig=<ID=chrUn_KI270312v1,length=998>\r\n",
      "##contig=<ID=chrUn_KI270311v1,length=12399>\r\n",
      "##contig=<ID=chrUn_KI270317v1,length=37690>\r\n",
      "##contig=<ID=chrUn_KI270412v1,length=1179>\r\n",
      "##contig=<ID=chrUn_KI270411v1,length=2646>\r\n",
      "##contig=<ID=chrUn_KI270414v1,length=2489>\r\n",
      "##contig=<ID=chrUn_KI270419v1,length=1029>\r\n",
      "##contig=<ID=chrUn_KI270418v1,length=2145>\r\n",
      "\r\n",
      "gzip: stdout: Broken pipe\r\n"
     ]
    }
   ],
   "source": [
    "!gunzip -c  /nrnb/users/btsui/Data/tcga_orig_vcf/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c/bcc7d112-2d5d-4ff1-a863-70f11d5b0f2c.vcf.gz | head -n 100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{\n",
      "  \"data\": {\n",
      "    \"data_type\": \"Raw Simple Somatic Mutation\",\n",
      "    \"updated_datetime\": \"2018-02-15T06:59:15.267328-06:00\",\n",
      "    \"created_datetime\": \"2017-12-01T01:01:39.435980-06:00\",\n",
      "    \"file_name\": \"9ad50bf6-9fe0-4ad7-b26f-b6f52824ef9e.vcf.gz\",\n",
      "    \"md5sum\": \"ff04b9d4d914e2f63a3fd376e6f0354b\",\n",
      "    \"data_format\": \"VCF\",\n",
      "    \"acl\": [\n",
      "      \"phs000178\"\n",
      "    ],\n",
      "    \"access\": \"controlled\",\n",
      "    \"state\": \"live\",\n",
      "    \"file_id\": \"9ad50bf6-9fe0-4ad7-b26f-b6f52824ef9e\",\n",
      "    \"data_category\": \"Simple Nucleotide Variation\",\n",
      "    \"file_size\": 44664,\n",
      "    \"submitter_id\": \"TCGA-TQ-A7RI-01A-11D-A33T-08_TCGA-TQ-A7RI-10A-01D-A33W-08_varscan_dr_10.0\",\n",
      "    \"type\": \"simple_somatic_mutation\",\n",
      "    \"file_state\": \"submitted\",\n",
      "    \"experimental_strategy\": \"WXS\"\n",
      "  },\n",
      "  \"warnings\": {}\n",
      "}\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Analysis ID\t\n",
    "#submitter_id might do, "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
